{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting pypdf\n",
      "  Downloading pypdf-3.2.1-py3-none-any.whl (237 kB)\n",
      "     -------------------------------------- 237.2/237.2 kB 1.2 MB/s eta 0:00:00\n",
      "Installing collected packages: pypdf\n",
      "Successfully installed pypdf-3.2.1\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install pypdf\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pypdf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "'ls' is not recognized as an internal or external command,\n",
      "operable program or batch file.\n"
     ]
    }
   ],
   "source": [
    "!ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Narrative review\n",
      "Machine learning for clinical decision support in infectious diseases: a\n",
      "narrative review of current applications\n",
      "N. Peiffer-Smadja1,2,*, T.M. Rawson1, R. Ahmad1, A. Buchard3, P. Georgiou4,\n",
      "F.-X. Lescure2,5, G. Birgand1, A.H. Holmes1\n",
      "1)National Institute for Health Research Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance, Imperi al College\n",
      "London, London, UK\n",
      "2)French Institute for Medical Research (Inserm), Infection Antimicrobials Modelling Evolution (IAME), UMR 1137, University Paris Diderot, Paris, France\n",
      "3)Babylon Health, London, UK\n",
      "4)Department of Electrical and Electronic Engineering, Imperial College, London, UK\n",
      "5)Infectious Diseases Department, Bichat-Claude Bernard Hospital, Assistance-Publique H ^opitaux de Paris, Paris, France\n",
      "article info\n",
      "Article history:\n",
      "Received 27 June 2019Received in revised form29 August 2019Accepted 9 September 2019Available online 17 September 2019\n",
      "Editor: A. Kalil\n",
      "Keywords:\n",
      "Artiﬁcial intelligence\n",
      "Clinical decision support systemInfectious diseasesInformation technologyMachine learningabstract\n",
      "Background: Machine learning (ML) is a growing ﬁeld in medicine. This narrative review describes the\n",
      "current body of literature on ML for clinical decision support in infectious diseases (ID).\n",
      "Objectives: We aim to inform clinicians about the use of ML for diagnosis, classi ﬁcation, outcome pre-\n",
      "diction and antimicrobial management in ID.Sources: References for this review were identi ﬁed through searches of MEDLINE/PubMed, EMBASE,\n",
      "Google Scholar, biorXiv, ACM Digital Library, arXiV and IEEE Xplore Digital Library up to July 2019.\n",
      "Content: We found 60 unique ML-clinical decision support systems (ML-CDSS) aiming to assist ID cli-\n",
      "nicians. Overall, 37 (62%) focused on bacterial infections, 10 (17%) on viral infections, nine (15%) ontuberculosis and four (7%) on any kind of infection. Among them, 20 (33%) addressed the diagnosis of\n",
      "infection, 18 (30%) the prediction, early detection or strati ﬁcation of sepsis, 13 (22%) the prediction of\n",
      "treatment response, four (7%) the prediction of antibiotic resistance, three (5%) the choice of antibioticregimen and two (3%) the choice of a combination antiretroviral therapy. The ML-CDSS were developed\n",
      "for intensive care units ( n¼24, 40%), ID consultation ( n¼15, 25%), medical or surgical wards ( n¼13,\n",
      "20%), emergency department ( n¼4, 7%), primary care ( n¼3, 5%) and antimicrobial stewardship ( n¼1,\n",
      "2%). Fifty-three ML-CDSS (88%) were developed using data from high-income countries and seven (12%)with data from low- and middle-income countries (LMIC). The evaluation of ML-CDSS was limited to\n",
      "measures of performance (e.g. sensitivity, speci ﬁcity) for 57 ML-CDSS (95%) and included data in clinical\n",
      "practice for three (5%).Implications: Considering comprehensive patient data from socioeconomically diverse healthcare set-\n",
      "tings, including primary care and LMICs, may improve the ability of ML-CDSS to suggest decisions\n",
      "adapted to various clinical contexts. Currents gaps identi ﬁed in the evaluation of ML-CDSS must also be\n",
      "addressed in order to know the potential impact of such tools for clinicians and patients. N. Peiffer-\n",
      "Smadja, Clin Microbiol Infect 2020;26:584\n",
      "©2019 European Society of Clinical Microbiology and Infectious Diseases. Published by Elsevier Ltd. All\n",
      "rights reserved.\n",
      "Introduction\n",
      "Artiﬁcial intelligence (AI), initiated in 1956, is often de ﬁned as\n",
      "the study of ‘intelligent agents ’, devices that perceive theirenvironment and take actions that maximize the likelihood of\n",
      "successfully achieving their goals [ 1]. AI in healthcare began with\n",
      "the development of expert systems, based on rules captured from\n",
      "interviews with medical experts, then translated and programmed\n",
      "[2]. With about 450 rules, MYCIN, the ﬁrst expert system in med-\n",
      "icine, was developed in 1976 and aimed to suggest antibiotic reg-\n",
      "imens for severe bacterial infections [ 3]. However, MYCIN was\n",
      "never actually used in practice because of the lack of system*Corresponding author. Nathan Peiffer-Smadja, National Institute for Health\n",
      "Research Health Protection Research Unit in Healthcare Associated Infections andAntimicrobial Resistance, Imperial College London, London, UK.\n",
      "E-mail address: n.peiffer-smadja@ic.ac.uk (N. Peiffer-Smadja).\n",
      "Contents lists available at ScienceDirect\n",
      "Clinical Microbiology and Infection\n",
      "journal homepage: www.clinicalmicrobiol ogyandinfection.com\n",
      "https://doi.org/10.1016/j.cmi.2019.09.009\n",
      "1198-743X/ ©2019 European Society of Clinical Microbiology and Infectious Diseases. Published by Elsevier Ltd. All rights reserved.Clinical Microbiology and Infection 26 (2020) 584 e595integration into clinical work. Another major limitation of expert\n",
      "systems is the large volume of rules needed to capture experts'\n",
      "knowledge in dif ﬁcult clinical decisions.\n",
      "Machine learning (ML) was developed to overcome the con-\n",
      "straints of expert systems [ 2]. In ML, engineers programme algo-\n",
      "rithms able to de ﬁne their own rules from data. Thus, human hand-\n",
      "coded rules are replaced by the arti ﬁcialﬁnding of rules from data.\n",
      "This allows ML systems to learn from data and interpret unknown\n",
      "situations. Among the panel of ML techniques developed, deep\n",
      "learning based on arti ﬁcial neural networks is the most famous [ 4].\n",
      "The performance and ability of the machine to learn is driven by the\n",
      "volume and quality of data provided, that is why ML systems are\n",
      "sometimes called data-intensive systems ( Fig. 1 ).\n",
      "The availability of healthcare data dramatically increased with\n",
      "electronic health record systems and the rise of connected devices\n",
      "[5]. The optimal analysis and interpretation of this important vol-\n",
      "ume of data, called ‘big data ’, needs the computing power of\n",
      "modern machines. In this context, ML tools integrating and making\n",
      "sense of huge amounts of complex data are becoming popular.\n",
      "Clinical decision support systems (CDSS) can be de ﬁned as software\n",
      "in which the characteristics of an individual patient are used topresent patient-speci ﬁc assessments or recommendations to the\n",
      "clinician towards a decision [ 6].Most current CDSS or computer-\n",
      "aided diagnosis or therapy are expert systems [ 7], but Machine\n",
      "Learning-Clinical Decision Support Systems (ML-CDSS) are drawing\n",
      "increased interest ( Fig. 2 )[8e10].\n",
      "ML systems have been developed in many ﬁelds of medicine,\n",
      "including radiology, with the interpretation of images from chest\n",
      "X-ray or magnetic resonance imaging for diagnostic purposes\n",
      "[11e15]. The ﬁrst FDA approval for an autonomous AI system\n",
      "took place in 2018 with IDx, a ML system used to detect diabetic\n",
      "retinopathy in retinal fundus photographs [ 16]. In infectious\n",
      "diseases (IDs), most ML work focuses on research, drug devel-\n",
      "opment or clinical microbiology. Systems have been developed to\n",
      "analyse bacterial genome and improve the prediction of resis-\n",
      "tance [ 17,18], HIV genotype and predict susceptibility to antire-\n",
      "troviral drugs [ 19], patterns of epidemics for surveillance\n",
      "purpose [ 20,21] or to discover new antibacterial drugs or vac-\n",
      "cines [ 22e24](Table 1 ).\n",
      "This article focuses on ML adapted to clinicians' decision (ML-\n",
      "CDSS) in ID. More precisely, we describe the objectives, character-istics, development and assessment of ML systems that may\n",
      "directly help clinicians with their work to diagnose infectious dis-\n",
      "eases, predict severity, decide whether they should use an anti-\n",
      "microbial and which antimicrobial to choose or what dosage.\n",
      "Material and methods\n",
      "Search strategy\n",
      "References for this review were identi ﬁed through searches of\n",
      "MEDLINE/PubMed, EMBASE, Google Scholar, biorXiv, ACM Digital\n",
      "Library, arXiV and IEEE Xplore Digital Library for articles by use of acombination of ML keywords ( ‘deep learning ’,‘artiﬁcial intelli-\n",
      "gence ’,‘artiﬁcial learning ’,‘machine learning ’,‘machine intelli-\n",
      "gence ’, ‘neural networks ’, ‘probabilistic networks ’, ‘knowledge\n",
      "representation ’,‘statistical learning ’,‘bayesian learning ’), decision-\n",
      "making keywords ( ‘medical decision ’,‘decision tool ’,‘support tool ’,\n",
      "‘clinical decision ’,‘physician decision ’,‘decision algorithm ’,‘CDSS ’,\n",
      "‘clinical management ’,‘decision making ’), antimicrobial keywords\n",
      "(‘antimicrobial ’,‘\n",
      "antibiotic ’,‘anti-infective ’,‘antifungal ’,‘antipara-\n",
      "sitic ’,‘antiviral ’) or sepsis keywords ( ‘sepsis ’,‘septic shock ’). We\n",
      "included articles resulting from these searches and relevant refer-\n",
      "ences cited in those articles up to July 2019.Study selection\n",
      "Prospective and retrospective articles in English that reported\n",
      "original research on ML-CDSS for ID were included. We included\n",
      "development reports, implementation studies, clinical trials or\n",
      "qualitative studies in primary, secondary, tertiary care including\n",
      "intensive care and paediatrics. We excluded studies that describe\n",
      "expert system CDSS, as de ﬁned by the use of manually pro-\n",
      "grammed rules, studies with ML systems that use data not\n",
      "currently available in routine clinical care (e.g. bacterial genome) or\n",
      "with outcomes irrelevant to clinicians' use for usual care (e.g.\n",
      "development of a new biomarker).\n",
      "Deﬁnitions of learning methods\n",
      "ML methods can be divided into supervised learning, unsuper-\n",
      "vised learning and reinforcement learning [ 2]. Supervised learning\n",
      "refers to algorithms using labelled data as a training dataset.\n",
      "Labelled data are data in which the outcome of interest is de ﬁned;\n",
      "for example, to train an algorithm for sepsis prediction, we use a\n",
      "dataset in which patients are already de ﬁned as having sepsis or\n",
      "not. The algorithm will then choose the best model to predict theoutcome of interest. In unsupervised learning, data are used\n",
      "without a prede ﬁned outcome of interest. Algorithms are left to\n",
      "their own to ﬁnd patterns and to extract hidden structure from data\n",
      "without any expert labelling. Unsupervised learning is mainly used\n",
      "in medicine to do clustering, aiming to discover inherent grouping\n",
      "in the data, such as similar groups of patients based on clinical data\n",
      "[25]. Reinforcement learning involves algorithms discovering ac-\n",
      "tions that yield the greatest rewards through trial and error. In this\n",
      "category, the algorithm is programmed to consider survival or a\n",
      "reduced hospital length of stay as a reward. A training dataset is\n",
      "used by the algorithm to conduct multiple tests in order to develop\n",
      "the model that obtains the highest reward [ 2,26].\n",
      "Analysis of data in the selected articles\n",
      "In each article, we extracted the variables that were analysed by\n",
      "the ML-CDSS. We split these variables into structured and un-\n",
      "structured variables. Structured variables are variables in which the\n",
      "possible values are prede ﬁned, e.g. for wound aspect, 0 ¼no ne-\n",
      "crosis, 1 ¼non-viable tissue, etc. Unstructured variables refer to\n",
      "free text such as a description of the wound by a clinician. We\n",
      "further split structured variables into demographic data (e.g. age,\n",
      "ethnicity), medical or surgical history (e.g. comorbidities, date of\n",
      "admission), vitals (e.g. temperature), symptoms or physical exam-\n",
      "ination ﬁndings (e.g. cough, lung auscultation), laboratory workup\n",
      "(e.g. creatinine), microbiology workup (e.g. antibiotic susceptibility\n",
      "testing), therapeutic history (e.g. type of surgery, administered\n",
      "drugs) or other workup (biopsy results, medical imaging). Medicalimaging can be considered as a structured variable, for example\n",
      "‘presence of lung in ﬁltrates ’can be mapped a priori to a uniquely\n",
      "deﬁned input value, or as an unstructured variable when the image\n",
      "itself is fed as input to the CDSS.\n",
      "Results\n",
      "General characteristics of ML-CDSS\n",
      "Among the 126 abstracts identi ﬁed and assessed for eligibility,\n",
      "14 were excluded because the variables were not available in\n",
      "routine clinical practice (e.g. bacterial genome), 29 because the\n",
      "CDSS was an expert system and eight because the outcome of the\n",
      "CDSS was not clinical (e.g. development of a novel biomarker).N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 585Fig. 1. Artiﬁcial intelligence for clinical infectious diseases.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 586In total, 75 papers with 60 unique ML-CDSS addressing clinical\n",
      "ID decisions were included. Thirty-one articles were found in\n",
      "MEDLINE/PubMed,12 in free repository of electronic preprints such\n",
      "as arXiv ( n¼8) or bioRxiv ( n¼4), ten in Google Scholar and ten in\n",
      "IEEE Xplore Digital Library. The main characteristics of the ML-CDSS\n",
      "are summarized in Table 2 and detailed in the Supplementary\n",
      "material .\n",
      "Overall, 37 (62%) ML-CDSS focused on bacterial infections, 10\n",
      "(17%) on viral infections, nine (15%) on tuberculosis and four (7%) on\n",
      "any kind of infection. Among them, 20 (33%) addressed the diag-\n",
      "nosis of infection, 18 (30%) the prediction, early detection or\n",
      "strati ﬁcation of sepsis, 13 (22%) the prediction of treatment\n",
      "response, four (7%) the prediction of antibiotic resistance, three\n",
      "(5%) the choice of antibiotic regimen and two (3%) the choice of a\n",
      "combination antiretroviral therapy (cART). Most of the ML-CDSS\n",
      "were developed with data from adult patients ( n¼54, 90%) but\n",
      "some targeted paediatric patients ( n¼3, 5%) [ 27e29] or neonates\n",
      "(n¼3, 5%) [ 30e32].\n",
      "The vast majority of ML-CDSS ( n¼58, 97%) used supervised\n",
      "learning whereas two (3%) used reinforcement learning [ 33,34].\n",
      "Prediction, early detection or strati ﬁcation of sepsis\n",
      "Eighteen ML-CDSS (30%) addressed the prediction, early detec-\n",
      "tion or strati ﬁcation of sepsis in the Intensive Care Unit (ICU).\n",
      "Among these ML-CDSS, 16 exclusively analysed structured patient\n",
      "variables: vitals ( n¼15), laboratory data ( n¼12), basic de-\n",
      "mographic information ( n¼7), medical history limited to main\n",
      "comorbidities and date of admission ( n¼7), therapeutic data\n",
      "(n¼5) and electrocardiogram waveform ( n¼1). Two ML-CDSS\n",
      "added unstructured clinical data to their model, one speci ﬁcally\n",
      "looked for antibiotic prescription in nursing notes to predict sepsis\n",
      "[35], but the other did not give details [ 36]. No CDSS for sepsis\n",
      "prediction used symptoms, physical examination ﬁndings nor\n",
      "microbiology data.\n",
      "All the 18 ML-CDSS were developed with data from high-income\n",
      "countries (HICs), with six ML-CDSS using the MIMIC (Multipa-\n",
      "rameter Intelligent Monitoring in Intensive Care) dataset. Thisdataset contains openly available data from ~50 000 critical care\n",
      "patients admitted to a Medical Center in Boston [ 37].Seventeen ML-CDSS were evaluated with measures of perfor-\n",
      "mance such as sensitivity or speci ﬁcity and one ML-CDSS was\n",
      "evaluated in the clinical setting. This ML-CDSS was developed by a\n",
      "company that published a set of papers describing its development,\n",
      "performance and impact in quasi-experimental studies [ 38]. They\n",
      "reported a relative reduction of in-hospital mortality between 30%\n",
      "and 60% after implementation of their ML-CDSS in ICUs [ 39e41]\n",
      "but do not use statistical methods adapted to quasi-experimental\n",
      "designs [ 42]. Indeed, they directly compare in-hospital mortality\n",
      "before and after the use of the ML-CDSS, a measure that is sus-\n",
      "ceptible to biases such as the history bias, i.e. the possibility that the\n",
      "change in the outcome may be linked to other changes in the\n",
      "environment rather than to the addition of the ML-CDSS [ 43]. They\n",
      "also published a randomized clinical trial in which 67 patients were\n",
      "randomized to a group monitored by a machine learning algorithm\n",
      "and 75 patients to a control group [ 44]. The trial reported a relative\n",
      "reduction of 58% of mortality in the group monitored by the ma-\n",
      "chine learning algorithm in which patients had blood cultures\n",
      "drawn and antibiotics administered approximately 2.8 hours before\n",
      "the control group.\n",
      "Diagnosis of infection\n",
      "Twenty ML-CDSS (33%) focused on the diagnosis of infection, to\n",
      "help clinicians decide if a patient has an infection, which infection\n",
      "(e.g. viral or bacterial) or an alternative non-infectious diagnosis.\n",
      "More precisely, six ML-CDSS address the diagnosis of tuberculosis\n",
      "(TB) in outpatient settings [ 45e50],ﬁve the diagnosis of bacterial\n",
      "infection in hospitalized patients [ 27,51e54], four the diagnosis of\n",
      "surgical site infection (SSI) [ 30,55e57], three the diagnosis of\n",
      "infection in emergency department [ 58e60] and two the distinc-\n",
      "tion between bacterial and viral meningitis [ 61,62]. Among the ﬁve\n",
      "ML-CDSS to diagnose bacterial infection in hospitalized patients,\n",
      "two ML-CDSS included any kind of infection [ 27,51], two focused on\n",
      "the prediction of positive blood cultures [ 53,54] and one on MRSA\n",
      "infection [ 52]. The ML-CDSS for patients hospitalized in surgical\n",
      "wards aimed to diagnose SSIs following open abdominal surgery\n",
      "[55,57] surgery for head or neck cancer [ 56] or following any\n",
      "intervention on neonates [ 30].\n",
      "Among the 20 ML-CDSS, 18 exclusively analysed structured\n",
      "patient variables: demographics data ( n¼15), medical history\n",
      "Fig. 2. Trends in the number of publications using selected search terms in PubMed in August 2019.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 587(n¼14), laboratory results ( n¼13), vitals ( n¼9), symptoms and\n",
      "physical examination ﬁndings ( n¼8), therapeutic history ( n¼7),\n",
      "chest X-ray for TB diagnosis ( n¼2) and microbiology data ( n¼2).\n",
      "Two ML-CDSS added free clinical text; nursing notes, clinical\n",
      "narrative or chief complaint [ 51,59]. Seventeen ML-CDSS were\n",
      "developed with data from HICs and three with data from low- and\n",
      "middle-income countries (LMICs), addressing the screening of TB in\n",
      "South Africa [ 48] or the diagnosis of TB in Turkey [ 49] or Iran [ 63].\n",
      "Nineteen ML-CDSS were evaluated by measures of sensitivity,\n",
      "speci ﬁcity and receiver ROC curves. One study described a ML-CDSS\n",
      "trained to diagnose bacterial infection using six routinely available\n",
      "blood parameters with data from 160 203 individuals and its use in\n",
      "a prospective observational cohort. Among 104 patients included,\n",
      "the ML-CDSS predicted a bacterial infection in three individuals\n",
      "who were not identi ﬁed by clinicians as having an infection on\n",
      "admission but were diagnosed later with a bacterial infection [ 58].\n",
      "No study assessed the clinical or microbiological impact of the use\n",
      "of ML-CDSS in clinical settings.\n",
      "Determinants of treatment outcomes\n",
      "Ten ML-CDSS (17%) addressed the prediction of treatment suc-\n",
      "cess for outpatients. Among them, ﬁve ML-CDSS predicted the\n",
      "virological response to HIV therapy [ 64e68] and three to HCVtherapy [ 69e71]. They all analysed therapeutic history but only\n",
      "four used viral genotype or medical history [ 65e67,71]. Therapeutic\n",
      "history was limited to previous HIV or HCV therapies but not to\n",
      "other drugs that the patient might have been taking. Two ML-CDSS\n",
      "predicted treatment outcome in TB using demographics data (e.g.education level, homelessness), TB history including constitutional\n",
      "symptoms, TB treatment and structured data from the chest\n",
      "radiograph (e.g. size of cavity) [ 72,73].\n",
      "Three ML-CDSS (5%) addressed the prediction of C. dif ﬁcilecolitis\n",
      "complications [ 74] or recurrence [ 75,76] in hospital settings. We\n",
      "found four ML-CDSS (7%) for the prediction of drug resistance: one\n",
      "predicted the risk of developing multidrug-resistant TB in Chinesepatients with TB [ 77] one combined demographic data and medical\n",
      "history for personalized prediction of baseline antibiotic resistance\n",
      "in urinary tract infection [ 78] one analysed demographic data,\n",
      "living conditions and treatment history of patients with positive\n",
      "blood cultures to predict baseline susceptibility to ampicillin, cef-\n",
      "triaxone and gentamicin [ 28] and one used ML to measure the\n",
      "impact of antibiotic exposure on the acquisition of colonization\n",
      "with extended-spectrum\n",
      "b-lactamase-producing Gram-negative\n",
      "bacteria [ 79].\n",
      "Thirteen ML-CDSS in this category were developed with patient\n",
      "data from HICs and two with patient data from LMICs to predict TB\n",
      "outcomes, one in Pakistan [ 72] and the other in Eastern Europe [ 73]Table 1\n",
      "List of reviews with a focus on machine learning in the ﬁeld of infectious diseases\n",
      "Topic Title Year of\n",
      "publicationReview\n",
      "typeDetailed outcome Settings Including CDSS\n",
      "Bacterial resistance Genome-based prediction of bacterial\n",
      "antibiotic resistance [ 18]2018 Narrative Analysis of bacterial genome to\n",
      "predict resistanceClinical microbiology\n",
      "labNo\n",
      "Machine learning: novel bioinformatics\n",
      "approaches for combating antimicrobialresistance [ 17]2017 Narrative Analysis of bacterial genome to\n",
      "predict resistanceClinical microbiology\n",
      "labNo\n",
      "Sepsis Emerging technologies for molecular\n",
      "diagnosis of sepsis [ 99]2018 Narrative Microbiological diagnosis of\n",
      "sepsisClinical microbiology\n",
      "labNo\n",
      "Prediction of sepsis ICU Yes\n",
      "HIV A survey of machine learning applications\n",
      "in HIV clinical research and care [ 100]2017 Systematic HIV/AIDS clinical research and\n",
      "medical care studies that utilizemachine learning methodologyMultiple settings Yes\n",
      "Computer-aided optimization of combined\n",
      "anti-retroviral therapy for HIV: new drugs,\n",
      "new drug targets and drug resistance [ 101]2016 Narrative Analysis of HIV genotype to\n",
      "predict drug susceptibility\n",
      "in vitro or response to\n",
      "combination antiretroviraltherapy in vivoVirology lab Yes\n",
      "Infection control Using online social networks to track a\n",
      "pandemic: A systematic review [ 102]2016 Systematic Analysis of online social\n",
      "network data to trackpandemicsHealth authorities No\n",
      "Automated surveillance of healthcare-\n",
      "associated infections: state of the art [ 21]2017 Narrative Surveillance of healthcare-\n",
      "associated infectionsInfection control unit No\n",
      "Introduction to machine learning in digital\n",
      "healthcare epidemiology [ 20]2018 Narrative Prediction, detection of trends\n",
      "and patterns for surveillancepurposesInfection control unit No\n",
      "Molecular biology Computational approaches for prediction of\n",
      "pathogen-host protein-protein interactions[103]2015 Narrative Prediction of pathogen-host\n",
      "protein-protein interactionResearch lab No\n",
      "Progress in computational studies of host-\n",
      "pathogen interactions [ 104]2013 Narrative Prediction of pathogen-host\n",
      "protein-protein interactionResearch lab No\n",
      "Identi ﬁcation of legionella effectors using\n",
      "bioinformatic approaches [ 105]2012 Narrative Identi ﬁcation\n",
      "ofL. pneumophila effectors\n",
      "Research lab No\n",
      "Vaccine development Comparative pathogenesis and systems\n",
      "biology for biodefense virus vaccinedevelopment [ 24]2010 Narrative Analysis of Virus-Host\n",
      "InteractionsResearch lab/Research\n",
      "and developmentNo\n",
      "Systems serology for evaluation of HIV\n",
      "vaccine trials [ 106]2017 Narrative De ﬁning humoral signatures in\n",
      "response to vaccinesResearch lab/Research\n",
      "and developmentNo\n",
      "Drug discovery Machine-learning techniques applied to\n",
      "antibacterial drug discovery [ 23]2014 Narrative Antibiotic drug discovery Research lab/Research\n",
      "and developmentNo\n",
      "Medical imaging Computer-assisted detection of infectious\n",
      "lung diseases: a review [ 107]2013 Narrative Analysis of medical imaging of\n",
      "respiratory tract infectionsRadiology department No\n",
      "Microscopy Microscopy in infectious disease research-\n",
      "imaging across scales [ 108]2018 Narrative Analysis of microscopic images\n",
      "in ID researchResearch lab NoN. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 588In addition, one ML-CDSS used data both from HICs and middle-\n",
      "income countries [ 79]. Another ML-CDSS was developed in HICs\n",
      "and then speci ﬁcally adapted to the speci ﬁcities of human immu-\n",
      "node ﬁciency virus (HIV) care in LMICs such as limited access to\n",
      "genotype data, infrequent visits to clinics and restricted list of\n",
      "available drugs [ 67]. It used the HIV Drug Resistance Database that\n",
      "includes large datasets from patients around the world including\n",
      "sub-Saharan countries.\n",
      "The 17 ML-CDSS predicting treatment outcomes or antibiotic\n",
      "susceptibilities were evaluated by measures of sensitivity, speci-\n",
      "ﬁcity and receiver operating characteristic (ROC) curves.\n",
      "Treatment selection\n",
      "Five ML-CDSS (8%) addressed the choice of antimicrobials: three\n",
      "ML-CDSS aimed to ﬁnd the optimal antibiotic regimen [ 33,80,81]and two the optimal cART for HIV [ 34,82]. All the CDSS focused on\n",
      "the choice of an agent but did not give individual advice on the dose\n",
      "or duration.\n",
      "The two ML-CDSS to guide cART focused on outpatient settings:\n",
      "one analysed treatment history and treatment objectives but did\n",
      "not consider clinical or biological data [ 82], whereas the other used\n",
      "demographics, medical history, CD4 þcell count, viral load, geno-\n",
      "typic data and treatment history [ 34]. Among the three ML-CDSS\n",
      "for antibiotic management, one was adapted to ICUs [ 33]. another\n",
      "one to primary care [ 81] and the third one targeted antimicrobial\n",
      "stewardship teams [ 80]. Their use of patient data was heteroge-\n",
      "neous; only one ML-CDSS took into account the identi ﬁcation of the\n",
      "pathogen in blood culture results but not the antimicrobial sus-\n",
      "ceptibility testing [ 33], medical history was not used in two ML-\n",
      "CDSS [ 33,80] and limited to breastfeeding, pregnancy, allergy and\n",
      "kidney failure in the third [ 81]. They did not use the therapeutic\n",
      "history of patients, neither data from the physical examination nor\n",
      "local antibiotic resistance rates and were not adapted for a dynamic\n",
      "use over time.\n",
      "All of these ML-CDSS concerned HICs, using data from North\n",
      "American or European patients and only one ML-CDSS was evalu-ated in a clinical setting. It was a ML-CDSS focusing on inappro-\n",
      "priate prescriptions for review by antimicrobial stewardship\n",
      "pharmacists. The addition of a ML module to an expert system was\n",
      "shown to identify inappropriate prescriptions of piperacillin-\n",
      "tazobactam that were missed by the expert system [ 80]. Howev-\n",
      "er, this study does not report the clinical or microbiological out-\n",
      "comes of this intervention.\n",
      "DiscussionHealthcare settings\n",
      "Most ML-CDSS found in the literature were developed for sec-\n",
      "ondary and tertiary care ( n¼57, 95%) whereas only three ML-CDSS\n",
      "(5%) targeted primary care [ 48,78,81]. Four ML-CDSS were specif-\n",
      "ically tailored for emergency care [ 46,58e60], compared with more\n",
      "than 20 ML-CDSS for ICUs. Moreover, only seven ML-CDSS (12%)\n",
      "were adapted to LMICs [ 48,49,63,67,72,73,79], six of them con-\n",
      "cerning HIV or TB.\n",
      "ML techniques usually need a large amount of data (e.g. more\n",
      "than 128 000 retinal images in a recent ophthalmologic ML sys-\n",
      "tem) [ 83]. Thus, it is no surprise that most ML-CDSS in ID currently\n",
      "should focus on domains generating high-quality databases\n",
      "including ICU and HIV patients from HICs. The MIMIC dataset was\n",
      "a recurrent data source for ML systems predicting sepsis [ 37]. In\n",
      "the same way, ML-CDSS targeting HIV therapy selection or the\n",
      "prediction of virological response were also based on large open-\n",
      "access databases [ 66]. The development of ML-CDSS is more\n",
      "difﬁcult in primary care where available patient data for the\n",
      "clinician is limited and databases are scarce. The same statement\n",
      "can be made for LMICs where data extraction is dif ﬁcult due to the\n",
      "frequent lack of clinical information system. The availability of\n",
      "data is also correlated with healthcare access: vulnerable pop-ulations tend to have a lower access to the healthcare system with\n",
      "a more fragmented care. This could undermine the ability of ML-\n",
      "CDSS to make adequate predictions for patients from vulnerable\n",
      "populations and lead to a potential increase in healthcare in-\n",
      "equalities [ 84,85].\n",
      "The unequal spreading of ML-CDSS across healthcare settings\n",
      "reﬂects the unequal availability of high-quality and large clinical\n",
      "databases. This is why a special effort should be made to target the\n",
      "development of such databases in underfunded settings such as\n",
      "primary care or LMICs and to include data from diverse settings and\n",
      "populations in training and validation datasets. Global open accessTable 2\n",
      "Summary of machine learning clinical decision support systems\n",
      "ML-CDSS characteristics n¼60 (%)\n",
      "Medical settingsa\n",
      "Intensive Care Unit 24 (40)\n",
      "Infectious diseases consultation 15 (25)Medical ward 9 (15)Surgical ward 5 (8)Emergency department 4 (7)\n",
      "Primary care 3 (5)\n",
      "Antimicrobial stewardship team 1 (2)\n",
      "Geographical settings\n",
      "a\n",
      "High-income countries 54 (90)Low- and middle-income countries 7 (12)\n",
      "Population\n",
      "Adults 53 (88)Neonates 3 (5)\n",
      "Paediatric patients 3 (5)\n",
      "Retirement-home 1 (2)\n",
      "Types of decision support\n",
      "Diagnostic\n",
      "Diagnosis of infection 20 (33)Prediction of sepsis 18 (30)Prediction of antibiotic resistance 4 (7)\n",
      "Therapeutic\n",
      "Prediction of treatment response 13 (22)\n",
      "Antibiotic selection 3 (5)HIV therapy selection 2 (3)\n",
      "Type of infection\n",
      "Bacterial infection 37 (62)Viral infection 10 (17)Mycobacterial infection 9 (15)Any kind of infection 4 (7)\n",
      "Types of learning\n",
      "Supervised learning 58 (97)Reinforcement learning 2 (3)\n",
      "Data\n",
      "Clinical data (e.g. demographics, vitals) 52 (87)Laboratory 38 (63)Therapy 28 (47)\n",
      "Microbiology 15 (25)\n",
      "Other workup (e.g. ECG, imaging) 6 (10)Unstructured clinical data (free text) 5 (8)\n",
      "Evaluation\n",
      "b\n",
      "Performance 57 (95)Use 3 (5)Adoption 0\n",
      "ML, machine learning; CDSS, clinical decision support systems; HIV, human im-\n",
      "munode ﬁciency virus.\n",
      "aOne article included patients in medical wards, surgical wards and intensive\n",
      "care units in high and middle-income countries.\n",
      "bWe separated studies that describe the performance (e.g. receiver operating\n",
      "characteristic curves) of the ML-CDSS, studies that describe the use of the CDSS inreal-life settings and studies that describe the adoption of the CDSS in routineclinical practice.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 589to these databases should be a requirement for future funding in\n",
      "order to increase the opportunities for ML-CDSS development\n",
      "everywhere in the world. Moreover, we have not found any study\n",
      "focusing on fungal or parasitic diseases in this review. These areas\n",
      "may represent interesting opportunities for developing CDSS using\n",
      "machine learning for diagnosis, prediction of resistance or choice of\n",
      "antifungal or antiparasitic therapy. The analysis of the outputs of\n",
      "ML-CDSS at the population level should also be encouraged as it\n",
      "may be useful for public health and research institutes to regularly\n",
      "monitor trends in diagnosis and prescribing and correct potential\n",
      "biases.\n",
      "If these constraints and potential issues are considered, ML-\n",
      "CDSS appear as credible options for supporting infection manage-\n",
      "ment in countries where ID specialists and antimicrobial teams\n",
      "may not be readily available [ 86].\n",
      "Choice of patient variables\n",
      "In the present review, many ML-CDSS did not include some\n",
      "clinically relevant data. Medical history analysed in ML-CDSS was\n",
      "often limited to the presence of two or three prede ﬁned diseases\n",
      "(kidney failure, cirrhosis, etc.) [ 56,87]. No ML-CDSS for the diag-\n",
      "nosis of infection considered patients exposed to infected in-\n",
      "dividuals or high-risk environment (e.g. travel history). We did not\n",
      "ﬁnd any ML-CDSS predicting sepsis and taking into account the\n",
      "results from blood cultures or from any other microbiology sample.\n",
      "Among the 60 ML-CDSS, chest X-Ray was the only medical imaging\n",
      "analysed [ 46,48,72]. No ML-CDSS to predict antibiotic resistance or\n",
      "to select an antibiotic regimen included data from local antibiotic\n",
      "resistance rates. Paradoxically, ML-CDSS often use less data than a\n",
      "human clinician would do. In some articles, the authors analysed\n",
      "the performance of ML-CDSS when using a reduced set of variables\n",
      "and they found that the sensitivity and speci ﬁcity of ML-CDSS were\n",
      "systematically better when they used a larger panel of variables\n",
      "[60,87,48]. especially when adding unstructured data [ 59,36].\n",
      "Ensuring integration of physical examination or detailed medical\n",
      "history in these systems is of critical importance. Progress in Nat-\n",
      "ural Language Processing (NLP, i.e. the ability of computers to\n",
      "analyse human language) may help the integration of free medicaltext in future decision systems [ 88]. Nonetheless, the development\n",
      "of ML-CDSS using minimal variables may be of interest when data\n",
      "are not available across some areas or in resource limited settings. A\n",
      "particular attention should be paid to which variables are used by\n",
      "the ML-CDSS to predict their outcome: e.g. we found a ML-CDSS\n",
      "that used the prescription of antibiotics in ICU to predict sepsis\n",
      "[35], which could provide good performance but seems clinically\n",
      "irrelevant.\n",
      "ML-CDSS are constrained by the quality and availability of the\n",
      "clinical data used for their development and validation. To ensure\n",
      "that future ML tools are useful for clinicians, efforts should be made\n",
      "to provide comprehensive databases that include relevant clinical\n",
      "data.\n",
      "Evaluation and use of ML-CDSS\n",
      "The evaluation of ML-CDSS in clinical ID is still lacking with 57\n",
      "ML-CDSS (95%) reporting measures of performance such as sensi-\n",
      "tivity and speci ﬁcity and only three (5%) evaluated in clinical\n",
      "practice. The validation of the performance in the original database\n",
      "should preferably be done in a prospective database collected for\n",
      "validation purpose in another location. Then, pilot studies testing\n",
      "the usability and usefulness of ML-CDSS in clinical settings should\n",
      "be conducted, followed by clinical trials, preferably randomized,\n",
      "evaluating patients' outcomes, process improvement or cost-\n",
      "effectiveness.\n",
      "Special attention should be paid to the integration and\n",
      "implementation of systems into clinical practice, and their adop-\n",
      "tion and utilization by clinicians [ 89]. Co-design including clini-\n",
      "cians, pharmacists engineers, data managers and service users\n",
      "may be key for success and may allow to study the interaction\n",
      "between CDSS and healthcare professionals [ 90]. The place of ML-\n",
      "CDSS in the work ﬂow of clinicians has to be considered as\n",
      "different clinical tasks are likely to require different degrees of\n",
      "human involvement ( Fig. 3 )[10]. Data should be easily entered\n",
      "into the CDSS, ideally by automatic extraction from the EHR, and\n",
      "daily refreshed with new patient information. The CDSS need to ﬁt\n",
      "into the complex medical decision-making pathway and support\n",
      "clinicians at different stages sometimes by presenting different\n",
      "Fig. 3. Interaction between clinicians and machine-learning systems according to automation level.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 590aspects of data [ 91]. According to the numerous limits that we\n",
      "found concerning ML-CDSS in ID, the replacement of physicians\n",
      "by ML tools seems highly unlikely. AI is best suited to augmenting\n",
      "human intelligence, bringing big-data into focus to support hu-\n",
      "man decision-making and developers should aim to emphasize\n",
      "the synergy and interaction between physicians and algorithms\n",
      "instead of reporting less interesting clinical-versus-algorithm\n",
      "comparison [ 8,92,93] Clinicians' expectations are that ML-CDSS\n",
      "will relieve them of repetitive tasks that need to be performed\n",
      "without errors and allow them to focus on tasks that require\n",
      "human skills: analysis of the broader context, abstraction capa-\n",
      "bility but also exercising empathy and building relationships.\n",
      "However, we have not found any in-depth describing of the\n",
      "interaction of physicians with ML-CDSS in real-world settings.\n",
      "Implementation research may help to take into account the global\n",
      "context in which ML-CDSS are used [ 7].\n",
      "Unexpected consequences of the use of ML also have to be\n",
      "included in the research agenda. Consequences commonly cited are\n",
      "the risk of ‘deskilling ’clinicians who use ML-CDSS routinely, the\n",
      "risk of a blind obedience to the CDSS or the risk of decreased\n",
      "interaction between the clinician and the patient. Studies haveshown that the development of the electronic health record (EHR)\n",
      "shifted clinicians' time from the patient to the computer and ML\n",
      "systems may represent an additional risk to do so [ 94,95]. By\n",
      "deﬁnition, unexpected consequences cannot be fully anticipated\n",
      "and research on ML-CDSS in real-world settings should aim to\n",
      "collect information on consequences both at the clinician and pa-\n",
      "tient -levels.Perspectives\n",
      "InTable 3 we summarize some essential criteria for future ML-\n",
      "CDSS adapted for clinicians' use. Clinicians must know when and\n",
      "why errors by ML in clinical-decision predictions might occur and\n",
      "when they cannot rely on an automated system whose intrinsic\n",
      "mechanisms they do not understand (the so-called ‘black box ’)[8].\n",
      "Visualization tools highlighting the variable or combination of\n",
      "variables that determined the output ought to be available for the\n",
      "prescriber and are an important parameter in future ML-CDSS\n",
      "[28,96]. ML-CDSS should also detail the trade-off that they apply\n",
      "between risk-taking and caution and show the con ﬁdence interval\n",
      "of their decisions and suggestions [ 9]. The impact of under- and\n",
      "overdiagnosis or treatment is not the same when considering up-\n",
      "per respiratory tract infection or sepsis therapy. Besides system\n",
      "improvement, we need to encourage and fund the education of\n",
      "clinicians about AI and ML in order to develop an AI-literate\n",
      "workforce in healthcare worldwide.\n",
      "In addition, global challenges for the development of arti ﬁcial\n",
      "intelligence and ML for clinical decisions are to be considered.\n",
      "Table 4 describes the main challenges for ML-CDSS that have been\n",
      "identi ﬁed in the literature. For example, controls have to ensure\n",
      "that ML-CDSS are registered on public databases, thoroughly\n",
      "evaluated independently of economic interest and validated using\n",
      "standardized frameworks. The privacy protection of data [ 97], cost-\n",
      "effectiveness research at the organizational and global level [ 89,98],\n",
      "and ethics discussion about accountability and liability when using\n",
      "ML-CDSS are other potential challenges to deal with.\n",
      "Table 3\n",
      "Necessary requirements for clinical decision support systems using machine learning\n",
      "Target Type of decision A clinically meaningful decision should be targeted, for example the diagnosis of a bacterial infection requiring antibioti cs or the\n",
      "choice of an antibiotic regimen\n",
      "Justiﬁcation of ML The use of machine learning instead of expert systems should be justi ﬁed, for example by the analysis of big data that could not\n",
      "be captured by expert systems.\n",
      "settings ML-CDSS should not only be developed for secondary or tertiary care but should also target such primary care prescribers as GPs\n",
      "or nurse prescribersThe ML-CDSS should offer a good performance in diverse settings including low- and middle-income countries\n",
      "Patients The ML-CDSS should be performant for different populations of patients like patients with diabetes, chronic kidney disease,\n",
      "cirrhosis, etc.If some comorbidities prevent the use of the ML-CDSS, it should be clearly stated and explained\n",
      "Users Different practitioners may not need the same tools even for the same decision. For example, surgeons do not have the same\n",
      "need for ML-CDSS addressing antimicrobial management as antimicrobial stewardship specialists or nurse prescribers inprimary care\n",
      "Data Comprehensiveness ML-CDSS should be able to analyse data from different sources such as structured clinical data, vitals, laboratory data,\n",
      "therapeutic history but also unstructured clinical data (e.g. free medical text, medical imaging)\n",
      "Availability ML-CDSS should not require data that are not available for most patients\n",
      "Extraction ML-CDSS should be able to automatically extract data from the electronic health record (EHR)\n",
      "Decision Uncertainty A way to present uncertainty should be found by the developers: e.g. it could give a con ﬁdence interval for the diagnosis of\n",
      "bacterial infection or a number of suggestions for antimicrobial therapy with associated probabilities of success according to itscalculation\n",
      "Clarity ML-CDSS should display what parameters were the most important for the suggested diagnosis or therapeutic decisionFlexibility Clinicians should be able to take out a speci ﬁc clinical or laboratory element that they do not think reliable and thus have access\n",
      "to modi ﬁed suggestions. They should also be able to add or emphasize elements that they ﬁnd particularly important or reliable.\n",
      "Memory Clinicians need ML-CDSS that can keep track of patients and for whom the data can be updated to modify the suggestions of the\n",
      "system\n",
      "Limits ML-CDSS should display very clearly what parameters have been used or not, e.g. medical history, medical imaging, etc. and\n",
      "other potential limits of its decision\n",
      "Interface Integration The ML-CDSS should ideally be integrated in the EHR used by the prescriber. A special attention should be paid to the\n",
      "interoperability of ML-CDSS across different EHR systems\n",
      "Speed The ML-CDSS should be quick to use, should automatically extract pertinent data from the EHR and give its output immediatelyUser-friendly Clinicians should be able to use ML-CDSS without speci ﬁc training\n",
      "Standardization As multiple ML-CDSS may be available for different diseases and conditions, interfaces for ML-CDSS should be standardized to a\n",
      "common format to increase usability. We should avoid having a software for every decision, e.g. diagnosis, antimicrobialprescribing, dosing, prevention of venous thromboembolic disease, etc.\n",
      "ML-CDSS, machine learning clinical decision support systems.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 591Table 4\n",
      "Global challenges for machine learning clinical decision support systems\n",
      "Challenges Description Potential solutions\n",
      "Data quantity There is a need for a huge quantity of clinical data to develop a machine\n",
      "learning (ML) system, especially for unsupervised MLSharable/open-source database such as MIMIC-III, a freely accessible\n",
      "critical care database that has helped to develop many ML-CDSS forsepsis in intensive care units\n",
      "Data quality ML-clinical decision support systems (CDSS) developers need to have\n",
      "access to high-quality data for training and validation dataset, especiallyfor supervised MLElectronic health record data standardization\n",
      "Pre-processing of data should be detailed in the description of ML-CCDSS\n",
      "Comprehensiveness ML-CDSS must be able to analyse data from different sources such as\n",
      "structured clinical data, vitals, laboratory data but also unstructured freetext (e.g. physical examination, medical history, etc.).Developers should try to include all the sources of data that are used by\n",
      "clinicians to take clinical decisionClinicians should be included during development to make sure thatunstructured data are not left out in ML-CDSS (co-design)\n",
      "Availability The data that are analysed by the system should be available for most\n",
      "patients (e.g. a CDSS analysing genetic data could not be used in usualcare)Importance of co-design with users, physicians, other prescribers and\n",
      "clinical managers to know what data is available for patientsPossibility of adapting the ML-CDSS to different settings in whichavailable data could be different\n",
      "Interoperability A multitude of technology platforms are involved in the generation,\n",
      "collection and mobilization of health data. ML-CDSS should be able tohave good performance with different platforms and electronic healthrecord systems.Initiatives as FHIR (Fast Healthcare Interoperability Resources) that\n",
      "standardize healthcare data to common format\n",
      "Equity Most ML-CDSS are data-driven. As patients with low socioeconomic\n",
      "status usually have a limited access to healthcare, less data from thesepatients are available and ML systems could create new inequalities orincrease health disparities. Patients with comorbidities should also be\n",
      "included in ML-CDSS.Developing ML-CDSS with training and validation datasets from\n",
      "socioeconomically diverse health care systemsEnsuring that patients with comorbidities (e.g. chronic kidney disease,cirrhosis) are included in the training dataset\n",
      "Ensuring that social determinants of health are being considered in the\n",
      "support systemTest ML-CDSS for potential discriminatory behaviour\n",
      "Adaptability ML-CDSS should be usable in different clinical settings including\n",
      "primary, secondary and tertiary care. A special attention should be paidto low- and middle-income countries that should not be left out byprogress in the development of AI.Diverse datasets coming from different countries\n",
      "Speci ﬁcities of LMICs should be considered early in the development of\n",
      "ML-CDSSAvailability of antimicrobials in every healthcare setting should beconsidered\n",
      "Interpretability Clinicians should be able to understand the output of ML-CDSS and their\n",
      "determinants for an intelligent use and interpretationEducation of an AI-literate clinician workforce (e.g. training in\n",
      "informatics and data science by the National Library of Medicine)Highlighting the variable or combination of variables that determinedthe output (e.g. ‘saliency map ’)\n",
      "Reporting of con ﬁdence level in the prediction\n",
      "Security Alternatives should be developed to make sure that healthcare systems\n",
      "could function normally in case of an electronic problem. ML-CDSS\n",
      "should also be protected against hacking and other ill-willed practicesSystematic debugging\n",
      "Security software\n",
      "Anticipation and preparedness\n",
      "Validation ML-CDSS should be validated by an independent organization. Both the\n",
      "risk of under-regulation and over-regulation should be consideredQuality standards, precerti ﬁcation and certi ﬁcation\n",
      "Such regulatory framework as the Software as Medical Device (SaMD)framework developed by the International Medical Device RegulatorsForum (IMDRF) working group chaired by the Food and DrugAdministration (FDA)\n",
      "Conﬁdence ML-CDSS developers should pay special attention to the con ﬁdence of\n",
      "clinicians and patients in arti ﬁcial intelligence. Limited con ﬁdence or\n",
      "fear of the users would limit the use of ML and blind con ﬁdence also\n",
      "called automation bias could be dangerousBehavioural science, sociological and anthropological research on\n",
      "artiﬁcial intelligence in medicine\n",
      "Training and education of clinicians about the bene ﬁts and limits of\n",
      "artiﬁcial intelligence and machine learning\n",
      "Studies on the interaction between machines and speci ﬁcally human\n",
      "skillsCo-design between engineers and clinicians\n",
      "Evaluation Evidence of the utility and bene ﬁts of ML-CDSS should not be limited to\n",
      "analysis of performance (e.g. ROC curves) but should consider clinicaloutcomes. Ideally, ML-CDSS should be evaluated in real-world settings\n",
      "from pilot study to clinical impact in real-world settingsStudies should not limit their outcomes to the evaluation of\n",
      "performance but should include clinical and microbiological outcomesRandomized controlled trial should try to determine the improvement\n",
      "of medical decision with the addition of ML-CDSS instead of comparing\n",
      "support systems with human clinicians\n",
      "Accountability The accountability of each actor (clinicians, ML-CDSS developers,\n",
      "certi\n",
      "ﬁcation organisms) should be de ﬁned beforehandResearch and forums on the accountability of AI and ML systems\n",
      "Inspiration from the work that has been done in other ﬁelds as\n",
      "driverless cars\n",
      "Cost Cost of ML-CDSS including development, data processing,\n",
      "implementation and impact on healthcare should be systematically\n",
      "studied, at the organizational and global levelCost-effectiveness analysis and research\n",
      "Privacy ML-CDSS should respect the privacy of health data and ensure proper\n",
      "consent and patient governance in data collectionGlobal regulation as the European Union General Data Protection\n",
      "Regulation or United States Health Insurance Portability andAccountability ActUse of anonymized or pseudonymized dataPrivacy audits\n",
      "Independence Regulation agencies should make sure that ML-CDSS are independent\n",
      "from drug and diagnostic test companies and do not favour any\n",
      "treatments or tests for economic purposesRegulatory frameworks should consider the need for independence\n",
      "Public funding for the development of AI and ML in healthcare\n",
      "Unexpected\n",
      "consequencesML-CDSS have the potential to deeply modify health systems and\n",
      "unexpected consequences should be systematically considered andstudiedSystematic research on the limits of the ML-CDSS\n",
      "Studying potentially unexpected consequences on clinicians like‘deskilling ’(loss of clinicians' skill due to the repeated use of ML-CDSS)\n",
      "or burn-outN. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 592Conclusion\n",
      "Current ML tools cover a range of clinical outcomes such as the\n",
      "prediction of sepsis in ICUs, the diagnosis of TB or SSI, the predic-\n",
      "tion of virological success of cART or the selection of an antibiotic\n",
      "regimen. However, 57 ML-CDSS out of 60 only reported perfor-\n",
      "mance measures such as sensitivity or speci ﬁcity and evidence is\n",
      "lacking regarding their use and impact in real-life clinical settings.\n",
      "Moreover, current ML-CDSS mainly use patient data from HICs,relying on large available open-access datasets and generally\n",
      "analyse a limited number of structured patient variables. Future\n",
      "ML-CDSS in ID should be developed in diverse health settings,\n",
      "including primary care and LMICs that are currently underrepre-\n",
      "sented, and be embedded in a structured process of integration into\n",
      "clinical settings. Future studies should aim to report clinical out-\n",
      "comes following sustainable use in routine clinical care.\n",
      "Transparency Declaration\n",
      "Dr Buchard reports that he is employed by Babylon Health. Dr\n",
      "Buchard has provided an expertise on machine learning methods\n",
      "exclusively on selected articles sent by the corresponding author.\n",
      "No tools developed by Babylon Health are discussed in the review.\n",
      "The authors have no other relevant con ﬂicts of interest to\n",
      "disclose.\n",
      "Funding\n",
      "The research was funded by a PhD studentship from ^Ile-de-\n",
      "France Regional Health Agency, Paris, France to Nathan Peiffer-\n",
      "Smadja, by a grant from the French Association de Chi-\n",
      "mioth /C19erapie Anti-Infectieuse (ACAI) to Nathan Peiffer-Smadja and\n",
      "by the National Institute for Health Research Health Protection\n",
      "Research Unit (NIHR HPRU) in Healthcare Associated Infection and\n",
      "Antimicrobial Resistance at Imperial College London in partnership\n",
      "with Public Health England (PHE). The views expressed are those of\n",
      "the author(s) and not necessarily those of the NHS, the NIHR, the\n",
      "Department of Health or Public Health England.\n",
      "Acknowledgements\n",
      "The support of the Economic and Social Research Council (ESRC)\n",
      "as part of the Antimicrobial Cross Council initiative supported by\n",
      "the seven UK research councils, and also the support of the Global\n",
      "Challenges Research Fund, are gratefully acknowledged.\n",
      "Appendix A. Supplementary data\n",
      "Supplementary data to this article can be found online at\n",
      "https://doi.org/10.1016/j.cmi.2019.09.009 .References\n",
      "[1]Crevier D. A.I: The tumultuous history of the search for arti ﬁcial intelligence.\n",
      "New York, NY, USA: Basic Books, Inc.; 1993 .\n",
      "[2]Mitchell TM. Machine learning. 1st ed. New York, NY, USA: McGraw-Hill,\n",
      "Inc.; 1997 .\n",
      "[3] Shortliffe EH. Computer-based medical consultations: mycin. Elsevier; 1976.\n",
      "https://doi.org/10.1016/B978-0-444-00179-5.50008-1 .\n",
      "[4] Hinton G. Deep learning - a technology with the potential to transform health\n",
      "care. JAMA 2018;320:1101 e2.https://doi.org/10.1001/jama.2018.11100 .\n",
      "[5] Beam AL, Kohane IS. Big data and machine learning in health care. JAMA\n",
      "2018;319:1317 e8.https://doi.org/10.1001/jama.2017.18391 .\n",
      "[6]Sim I, Gorman P, Greenes RA, Haynes RB, Kaplan B, Lehmann H, et al. Clinical\n",
      "decision support systems for the practice of evidence-based medicine. J Am\n",
      "Med Inform Assoc JAMIA 2001;8:527 e34.\n",
      "[7] Rawson TM, Moore LSP, Hernandez B, Charani E, Castro-Sanchez E, Herrero P,\n",
      "et al. A systematic review of clinical decision support systems for antimi-\n",
      "crobial management: are we failing to investigate these interventions\n",
      "appropriately? Clin Microbiol Infect 2017;23:524 e32. https://doi.org/\n",
      "10.1016/j.cmi.2017.02.028 .\n",
      "[8] Cabitza F, Rasoini R, Gensini GF. Unintended consequences of machine\n",
      "learning in medicine. JAMA 2017;318:517. https://doi.org/10.1001/\n",
      "jama.2017.7797 .\n",
      "[9] Challen R, Denny J, Pitt M, Gompels L, Edwards T, Tsaneva-Atanasova K.\n",
      "Artiﬁcial intelligence, bias and clinical safety. BMJ Qual Saf 2019. https://\n",
      "doi.org/10.1136/bmjqs-2018-008370 . bmjqs-2018-008370.\n",
      "[10] Rawson TM, Ahmad R, Toumazou C, Georgiou P, Holmes AH. Arti ﬁcial in-\n",
      "telligence can improve decision-making in infection management. Nat HumBehav 2019;3:543 e5.https://doi.org/10.1038/s41562-019-0583-9 .\n",
      "[11] Choy G, Khalilzadeh O, Michalski M, Do S, Samir AE, Pianykh OS, et al. Cur-\n",
      "rent applications and future impact of machine learning in radiology. Radi-\n",
      "ology 2018;288:318 e28.https://doi.org/10.1148/radiol.2018171820 .\n",
      "[12] Lindsey R, Daluiski A, Chopra S, Lachapelle A, Mozer M, Sicular S, et al. Deep\n",
      "neural network improves fracture detection by clinicians. Proc Natl Acad Sci\n",
      "2018;115:11591 e6.https://doi.org/10.1073/pnas.1806905115 .\n",
      "[13] Jha S, Topol EJ. Adapting to arti ﬁcial intelligence: radiologists and patholo-\n",
      "gists as information specialists. JAMA 2016;316:2353. https://doi.org/\n",
      "10.1001/jama.2016.17438 .\n",
      "[14] Esteva A, Kuprel B, Novoa RA, Ko J, Swetter SM, Blau HM, et al. Dermatolo-\n",
      "gist-level classi ﬁcation of skin cancer with deep neural networks. Nature\n",
      "2017;542:115 e8.https://doi.org/10.1038/nature21056 .\n",
      "[15] Topol EJ. High-performance medicine: the convergence of human and arti-\n",
      "ﬁcial intelligence. Nat Med 2019;25:44 e56.https://doi.org/10.1038/s41591-\n",
      "018-0300-7 .\n",
      "[16] Abr /C18\n",
      "amoff MD, Lavin PT, Birch M, Shah N, Folk JC. Pivotal trial of an auton-\n",
      "omous AI-based diagnostic system for detection of diabetic retinopathy in\n",
      "primary care of ﬁces. Npj Digit Med 2018;1:39. https://doi.org/10.1038/\n",
      "s41746-018-0040-6 .\n",
      "[17] Macesic N, Polubriaginof F, Tatonetti NP. Machine learning: novel bioinfor-\n",
      "matics approaches for combating antimicrobial resistance. Curr Opin Infect\n",
      "Dis 2017;30:511 e7.https://doi.org/10.1097/QCO.0000000000000406 .\n",
      "[18] Su M, Satola SW, Read TD. Genome-based prediction of bacterial antibiotic\n",
      "resistance. J Clin Microbiol 2018. https://doi.org/10.1128/JCM.01405-18 .\n",
      "[19] Zazzi M, Cozzi-Lepri A, Prosperi MCF. Computer-aided optimization of\n",
      "combined anti-retroviral therapy for HIV: new drugs, new drug targets and\n",
      "drug resistance. Curr HIV Res 2016;14:101 e9.https://doi.org/10.2174/\n",
      "1570162X13666151029102254 .\n",
      "[20] Roth JA, Battegay M, Juchler F, Vogt JE, Widmer AF. Introduction to machine\n",
      "learning in digital healthcare epidemiology. Infect Control Hosp Epidemiol\n",
      "2018;39:1457 e62.https://doi.org/10.1017/ice.2018.265 .\n",
      "[21] Sips ME, Bonten MJM, van Mourik MSM. Automated surveillance of\n",
      "healthcare-associated infections: state of the art. Curr Opin Infect Dis\n",
      "2017;30:425 e31.https://doi.org/10.1097/QCO.0000000000000376 .\n",
      "[22] Lee EY, Lee MW, Fulan BM, Ferguson AL, Wong GCL. What can machine\n",
      "learning do for antimicrobial peptides, and what can antimicrobial peptides\n",
      "do for machine learning? Interface Focus 2017;7:20160153. https://doi.org/\n",
      "10.1098/rsfs.2016.0153 .\n",
      "[23] Durrant JD, Amaro RE. Machine-learning techniques applied to antibacterial\n",
      "drug discovery. Chem Biol Drug Des 2015;85:14 e21. https://doi.org/\n",
      "10.1111/cbdd.12423 .Table 4 (continued )\n",
      "Challenges Description Potential solutions\n",
      "Implementation The implementation of ML-CDSS should be thought of and organized as\n",
      "early as possible in the developmentML-CDSS developing teams should include specialists from\n",
      "implementation scienceEvaluation should include research on implementation, adoption, useand sustainabilitySpeci ﬁc task force committees to deal with AI implementation issues\n",
      "may be useful for developing a common vision at a specialty-wide level\n",
      "Sustainability ML-CDSS should be sustainable and updated according to new research,\n",
      "data or regulationSpeci ﬁc and sustainable funding should be given to developers and\n",
      "adopters of ML-CDSSN. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 593[24] Bowick GC, Barrett ADT. Comparative pathogenesis and systems biology for\n",
      "biodefense virus vaccine development. J Biomed Biotechnol 2010;2010:\n",
      "236528. https://doi.org/10.1155/2010/236528 .\n",
      "[25] Deo RC. Machine learning in medicine. Circulation 2015;132:1920 e30.\n",
      "https://doi.org/10.1161/CIRCULATIONAHA.115.001593 .\n",
      "[26] Russell S, Norvig P. Arti ﬁcial intelligence: a modern approach. Third. Pear-\n",
      "son; 2016 .\n",
      "[27] Lamping F, Jack T, Rübsamen N, Sasse M, Beerbaum P, Mikolajczyk RT, et al.\n",
      "Development and validation of a diagnostic model for early differentiation of\n",
      "sepsis and non-infectious SIRS in critically ill children - a data-driven\n",
      "approach using machine-learning algorithms. BMC Pediatr 2018;18:112.\n",
      "https://doi.org/10.1186/s12887-018-1082-2 .\n",
      "[28] Oonsivilai M, Mo Y, Luangasanatip N, Lubell Y, Miliya T, Tan P, et al. Using\n",
      "machine learning to guide targeted and locally-tailored empiric antibiotic\n",
      "prescribing in a children ’s hospital in Cambodia. Wellcome Open Res 2018;3.\n",
      "https://doi.org/10.12688/wellcomeopenres.14847.1 .\n",
      "[29] Desautels T, Hoffman J, Barton C, Mao Q, Jay M, Calvert J, et al. Pediatric\n",
      "severe sepsis prediction using machine learning. bioRxiv 2017:223289.https://doi.org/10.1101/223289 .\n",
      "[30] Bartz-Kurycki MA, Green C, Anderson KT, Alder AC, Bucher BT, Cina RA, et al.\n",
      "Enhanced neonatal surgical site infection prediction model utilizing statis-\n",
      "tically and clinically signi ﬁcant variables in combination with a machine\n",
      "learning algorithm. Am J Surg 2018;216:764 e77.https://doi.org/10.1016/\n",
      "j.amjsurg.2018.07.041 .\n",
      "[31] Hu Y, Lee VCS, Tan K. Prediction of clinicians ’treatment in preterm infants\n",
      "with suspected late-onset sepsis - an ML approach. In: 2018 13th IEEE\n",
      "conference on industrial electronics and applications (ICIEA); 2018.\n",
      "p. 1177 e82.https://doi.org/10.1109/ICIEA.2018.8397888 .\n",
      "[32] Mani S, Ozdas A, Aliferis C, Varol HA, Chen Q, Carnevale R, et al. Medical\n",
      "decision support using machine learning for early detection of late-onsetneonatal sepsis. J Am Med Inform Assoc JAMIA 2014;21:326 e36.https://\n",
      "doi.org/10.1136/amiajnl-2013-001854 .\n",
      "[33] Tsoukalas A, Albertson T, Tagkopoulos I. From data to optimal decision\n",
      "making: a data-driven, probabilistic machine learning approach to decision\n",
      "support for patients with sepsis. JMIR Med Inform 2015;3:e11. https://\n",
      "doi.org/10.2196/medinform.3445 .\n",
      "[34] Parbhoo S, Bogojeska J, Zazzi M, Roth V, Doshi-Velez F. Combining kernel and\n",
      "model based learning for HIV therapy selection. AMIA Summits Transl Sci\n",
      "Proc 2017;2017:239 e48.\n",
      "[35] Apostolova E, Velez T. Toward automated early sepsis alerting: identifying\n",
      "infection patients from nursing notes. ArXiv180903995 Cs. 2018. http://\n",
      "arxiv.org/abs/1809.03995 . [Accessed 20 February 2019].\n",
      "[36] Culliton P, Levinson M, Ehresman A, Wherry J, Steingrub JS, Gallant SI. Pre-\n",
      "dicting severe sepsis using text from the electronic health record.\n",
      "ArXiv171111536 Cs. 2017. http://arxiv.org/abs/1711.11536 . [Accessed 30\n",
      "January 2019].\n",
      "[37] Johnson AEW, Pollard TJ, Shen L, Lehman LH, Feng M, Ghassemi M, et al.\n",
      "MIMIC-III, a freely accessible critical care database. Sci Data 2016;3:160035.\n",
      "https://doi.org/10.1038/sdata.2016.35 .\n",
      "[38] Calvert JS, Price DA, Chettipally UK, Barton CW, Feldman MD, Hoffman JL,\n",
      "et al. A computational approach to early sepsis detection. Comput Biol Med\n",
      "2016;74:69 e73.https://doi.org/10.1016/j.compbiomed.2016.05.003 .\n",
      "[39] McCoy A, Das R. Reducing patient mortality, length of stay and readmissions\n",
      "through machine learning-based sepsis prediction in the emergencydepartment, intensive care unit and hospital ﬂoor units. BMJ Open Qual\n",
      "2017;6:e000158. https://doi.org/10.1136/bmjoq-2017-000158\n",
      ".\n",
      "[40] Burdick H, Pino E, Gabel-Comeau D, Gu C, Huang H, Lynn-Palevsky A, et al.\n",
      "Evaluating a sepsis prediction machine learning algorithm in the emergency\n",
      "department and intensive care unit: a before and after comparative study.bioRxiv 2018. https://doi.org/10.1101/224014 .\n",
      "[41] Burdick H, Pino E, Gabel-Comeau D, McCoy A, Gu C, Roberts J, et al. Effect of a\n",
      "sepsis prediction algorithm on patient mortality, length of stay, and read-\n",
      "mission. bioRxiv 2018. https://doi.org/10.1101/457465 .\n",
      "[42] B €arnighausen T, Oldenburg C, Tugwell P, Bommer C, Ebert C, Barreto M, et al.\n",
      "Quasi-experimental study designs series-paper 7: assessing the assumptions.\n",
      "J Clin Epidemiol 2017;89:53 e66. https://doi.org/10.1016/j.jclinepi.2017.\n",
      "02.017 .\n",
      "[43] Ho AMH, Phelan R, Mizubuti GB, Murdoch JAC, Wickett S, Ho AK, et al. Bias in\n",
      "before-after studies: narrative overview for anesthesiologists. Anesth Analg\n",
      "2018;126:1755 e62.https://doi.org/10.1213/ANE.0000000000002705 .\n",
      "[44] Shimabukuro DW, Barton CW, Feldman MD, Mataraso SJ, Das R. Effect of a\n",
      "machine learning-based severe sepsis prediction algorithm on patient sur-\n",
      "vival and hospital length of stay: a randomised clinical trial. BMJ Open Respir\n",
      "Res 2017;4:e000234. https://doi.org/10.1136/bmjresp-2017-000234 .\n",
      "[45] Saybani MR, Shamshirband S, Golzari Hormozi S, Wah TY, Aghabozorgi S,\n",
      "Pourhoseingholi MA, et al. Diagnosing tuberculosis with a novel supportvector machine-based arti ﬁcial immune recognition system. Iran Red Cres-\n",
      "cent Med J 2015;17. https://doi.org/10.5812/ircmj.17(4)2015.24557 .\n",
      "[46] El-Solh AA, Hsiao C-B, Goodnough S, Serghani J, Grant BJB. Predicting active\n",
      "pulmonary tuberculosis using an arti ﬁcial neural network. Chest 1999;116:\n",
      "968e73.https://doi.org/10.1378/chest.116.4.968 .\n",
      "[47] Ansari AQ, Gupta NK, Ekata. Adaptive neurofuzzy system for tuberculosis. In:\n",
      "2012 2nd IEEE international conference on parallel, distributed and gridcomputing; 2012. p. 568 e73.https://doi.org/10.1109/PDGC.2012.6449883 .[48] Melendez J, S /C19anchez CI, Philipsen RHHM, Maduskar P, Dawson R, Theron G,\n",
      "et al. An automated tuberculosis screening strategy combining X-ray-based\n",
      "computer-aided detection and clinical information. Sci Rep 2016;6. https://\n",
      "doi.org/10.1038/srep25265 .\n",
      "[49] Er O, Temurtas F, Tanr ıkulu AÇ. Tuberculosis disease diagnosis using arti ﬁcial\n",
      "neural networks. J Med Syst 2010;34:299 e302. https://doi.org/10.1007/\n",
      "s10916-008-9241-x .\n",
      "[50] Santos AM, Pereira BB, Seixas JM, Mello FCQ, Kritski AL. Neural networks: an\n",
      "application for predicting smear negative pulmonary tuberculosis. In:\n",
      "Auget J-L, Balakrishnan N, Mesbah M, Molenberghs G, editors. Advances in\n",
      "statistical methods for the health sciences. Boston, MA: Birkh €auser Boston;\n",
      "2007. p. 275 e87.https://doi.org/10.1007/978-0-8176-4542-7_18 .\n",
      "[51] Revuelta-Zamorano P, S /C19anchez A, Rojo-/C19Alvarez JL,/C19Alvarez-Rodríguez J,\n",
      "Ramos-L /C19opez J, Soguero-Ruiz C. Prediction of healthcare associated in-\n",
      "fections in an intensive care unit using machine learning and big data tools.\n",
      "In: Kyriacou E, Christo ﬁdes S, Pattichis CS, editors. XIV mediterranean con-\n",
      "ference on medical and biological engineering and computing 2016, vol. 57.\n",
      "Cham: Springer International Publishing; 2016. p. 840 e5.https://doi.org/\n",
      "10.1007/978-3-319-32703-7_163 .\n",
      "[52] Hartvigsen T, Sen C, Brownell S, Teeple E, Kong X, Rundensteiner E. Early\n",
      "prediction of MRSA infections using electronic health records. In: Pro-\n",
      "ceedings of the 11th international joint conference on biomedical engi-\n",
      "neering systems and technologies; 2018. p. 156 e67.https://doi.org/10.5220/\n",
      "0006599601560167 .\n",
      "[53] Hernandez B, Herrero P, Rawson TM, Moore LSP, Evans B, Toumazou C,\n",
      "et al. Supervised learning for infection risk inference using pathology data.\n",
      "BMC Med Inform Decis Mak 2017;17. https://doi.org/10.1186/s12911-017-\n",
      "0550-1 .\n",
      "[54] Van Steenkiste T, Ruyssinck J, De Baets L, Decruyenaere J, De Turck F,\n",
      "Ongenae F, et al. Accurate prediction of blood culture outcome in theintensive care unit using long short-term memory neural networks. Artif\n",
      "Intell Med 2018. https://doi.org/10.1016/j.artmed.2018.10.008 .\n",
      "[55] Sanger PC, van Ramshorst GH, Mercan E, Huang S, Hartzler AL,\n",
      "Armstrong CAL, et al. A prognostic model of surgical site infection using daily\n",
      "clinical wound assessment. J Am Coll Surg 2016;223:259 e70.https://doi.org/\n",
      "10.1016/j.jamcollsurg.2016.04.046 . e2.\n",
      "[56] Kuo P-J, Wu S-C, Chien P-C, Chang S-S, Rau C-S, Tai H-L, et al. Arti ﬁcial neural\n",
      "network approach to predict surgical site infection after free- ﬂap recon-\n",
      "struction in patients receiving surgery for head and neck cancer. Oncotarget\n",
      "2018;9:13768 e82.https://doi.org/10.18632/oncotarget.24468 .\n",
      "[57] Ke C, Jin Y, Evans H, Lober B, Qian X, Liu J, et al. Prognostics of surgical site\n",
      "infections using dynamic health data. J Biomed Inform 2017;65:22 e33.\n",
      "https://doi.org/10.1016/j.jbi.2016.10.021 .\n",
      "[58] Rawson TM, Hernandez B, Moore LSP, Blandy O, Herrero P, Gilchrist M, et al.\n",
      "Supervised machine learning for the prediction of infection on admission to\n",
      "hospital: a prospective observational cohort study. J Antimicrob Chemother\n",
      "2019;74:1108 e15.https://doi.org/10.1093/jac/dky514 .\n",
      "[59] Horng S, Sontag DA, Halpern Y, Jernite Y, Shapiro NI, Nathanson LA. Creating\n",
      "an automated trigger for sepsis clinical decision support at emergency\n",
      "department triage using machine learning. Groza T, ed. PLoS One 2017;12:\n",
      "e0174708. https://doi.org/10.1371/journal.pone.0174708 .\n",
      "[60] Taylor RA, Moore CL, Cheung K-H, Brandt C. Predicting urinary tract in-\n",
      "fections in the emergency department with machine learning. PLoS One\n",
      "2018;13:e0194085. https://doi.org/10.1371/journal.pone.0194085 .\n",
      "[61] Revett K, Gorunescu F, Ene M. A machine learning approach to differenti-\n",
      "ating bacterial from viral meningitis. IEEE; 2006. p. 155 e62.https://doi.org/\n",
      "10.1109/JVA.2006.2 .\n",
      "[62] D ’Angelo G, Pilla R, Tascini C, Rampone S. A proposal for distinguishing be-\n",
      "tween bacterial and viral meningitis using genetic programming and deci-sion trees. Soft Comput 2019. https://doi.org/10.1007/s00500-018-03729-y .\n",
      "[63] Shamshirband S, Hessam S, Javidnia H, Amiribesheli M, Vahdat S, Petkovi /C19cD ,\n",
      "et al. Tuberculosis disease diagnosis using arti ﬁcial immune recognition\n",
      "system. Int J Med Sci 2014;11:508 e14.https://doi.org/10.7150/ijms.8249 .\n",
      "[64] Hatzakis GE, Mathur M, Gilbert L, Panos G, Wanchu A, Patel AK, et al. Neural\n",
      "network-longitudinal assessment of the Electronic Anti-Retroviral THerapy\n",
      "(EARTH) cohort to follow response to HIV-treatment. AMIA Annu Symp ProcAMIA Symp 2005:301 e5\n",
      ".\n",
      "[65] Beerenwinkel N, Lengauer T, D €aumer M, Kaiser R, Walter H, Korn K, et al.\n",
      "Methods for optimizing antiviral combination therapies. Bioinforma Oxf Engl\n",
      "2003;19:i16 e25.\n",
      "[66] Zazzi M, Incardona F, Rosen-Zvi M, Prosperi M, Lengauer T, Altmann A, et al.\n",
      "Predicting response to antiretroviral treatment by machine learning: the\n",
      "EuResist project. Intervirology 2012;55:123 e7.https://doi.org/10.1159/\n",
      "000332008 .\n",
      "[67] Revell AD, Wang D, Perez-Elias M-J, Wood R, Cogill D, Tempelman H, et al.\n",
      "2018 update to the HIV-TRePS system: the development of new computa-tional models to predict HIV treatment outcomes, with or without a geno-\n",
      "type, with enhanced usability for low-income settings. J Antimicrob\n",
      "Chemother 2018;73:2186 e96.https://doi.org/10.1093/jac/dky179 .\n",
      "[68] Petersen ML, LeDell E, Schwab J, Sarovar V, Gross R, Reynolds N, et al. Super\n",
      "learner analysis of electronic adherence data improves viral prediction and\n",
      "may provide strategies for selective HIV RNA monitoring. J Acquir Immune\n",
      "Deﬁc Syndr 1999 2015;69:109 e18. https://doi.org/10.1097/QAI.0000000\n",
      "000000548 .N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 594[69] Maiellaro PA, Cozzolongo R, Marino P. Arti ﬁcial neural networks for the\n",
      "prediction of response to interferon plus ribavirin treatment in patients with\n",
      "chronic hepatitis C. Curr Pharm Des 2004;10:2101 e9.\n",
      "[70] Takayama T, Ebinuma H, Tada S, Yamagishi Y, Wakabayashi K, Ojiro K, et al.\n",
      "Prediction of effect of pegylated interferon alpha-2b plus ribavirin combi-\n",
      "nation therapy in patients with chronic hepatitis C infection. PLoS One2011;6:e27223. https://doi.org/10.1371/journal.pone.0027223 .\n",
      "[71] Sargolzaee Aval F, Behnaz N, Raoufy MR, Alavian SM. Predicting the out-\n",
      "comes of combination therapy in patients with chronic hepatitis C using\n",
      "artiﬁcial neural network. Hepat Mon 2014;14:e17028. https://doi.org/\n",
      "10.5812/hepatmon.17028 .\n",
      "[72] Hussain OA, Junejo KN. Predicting treatment outcome of drug-susceptible\n",
      "tuberculosis patients using machine-learning models. Inform Health SocCare 2018:1 e17.https://doi.org/10.1080/17538157.2018.1433676 .\n",
      "[73] Sauer CM, Sasson D, Paik KE, McCague N, Celi LA, S /C19anchez Fern /C19andez I, et al.\n",
      "Feature selection and prediction of treatment failure in tuberculosis. Huang\n",
      "Z, ed. PLoS One 2018;13:e0207491. https://doi.org/10.1371/journal.pone.\n",
      "0207491 .\n",
      "[74] Li BY, Oh J, Young VB, Rao K, Wiens J. Using machine learning and the\n",
      "electronic health record to predict complicated Clostridium dif ﬁcile infec-\n",
      "tion. Open Forum Infect Dis 2019;6:ofz186. https://doi.org/10.1093/o ﬁd/\n",
      "ofz186 .\n",
      "[75] LaBarbera FD, Nikiforov I, Parvathenani A, Pramil V, Gorrepati S. A prediction\n",
      "model for Clostridium dif ﬁcile recurrence. J Community Hosp Intern Med\n",
      "Perspect 2015;5:26033. https://doi.org/10.3402/jchimp.v5.26033 .\n",
      "[76] Escobar GJ, Baker JM, Kipnis P, Greene JD, Mast TC, Gupta SB, et al. Prediction\n",
      "of recurrent Clostridium dif ﬁcile infection using comprehensive electronic\n",
      "medical records in an integrated healthcare delivery system. Infect Control\n",
      "Hosp Epidemiol 2017;38:1196 e203. https://doi.org/10.1017/ice.2017.176 .\n",
      "[77] Li S, Tang B, He H. An imbalanced learning based MDR-TB early warning\n",
      "system. J Med Syst 2016;40. https://doi.org/10.1007/s10916-016-0517-2 .\n",
      "[78] Yelin I, Snitser O, Novich G, Katz R, Tal O, Parizade M, et al. Personal clinical\n",
      "history predicts antibiotic resistance of urinary tract infections. Nat Med\n",
      "2019;25:1143 e52.https://doi.org/10.1038/s41591-019-0503-6 .\n",
      "[79] Tacconelli E, G /C19orska A, De Angelis G, Lammens C, Restuccia G, Schrenzel J,\n",
      "et al. Estimating the association between antibiotic exposure and coloniza-\n",
      "tion with extended-spectrum\n",
      "b-lactamase-producing Gram-negative bacte-\n",
      "ria using machine learning methods: a multicentre, prospective cohort\n",
      "study. Clin Microbiol Infect 2020;26:87 e94.\n",
      "[80] Beaudoin M, Kabanza F, Nault V, Valiquette L. Evaluation of a machine\n",
      "learning capability for a clinical decision support system to enhance anti-\n",
      "microbial stewardship programs. Artif Intell Med 2016;68:29 e36.https://\n",
      "doi.org/10.1016/j.artmed.2016.02.001 .\n",
      "[81] Shen Y, Yuan K, Chen D, Colloc J, Yang M, Li Y, et al. An ontology-driven clinical\n",
      "decision support system (IDDAP) for infectious disease diagnosis and antibiotic\n",
      "prescription. Artif Intell Med 2018;86:20 e32. https://doi.org/10.1016/\n",
      "j.artmed.2018.01.003 .\n",
      "[82] Ying Hao, Lin Feng, MacArthur Rodger D, Cohn Jonathan A, Barth-\n",
      "Jones Daniel C, Ye Hong, et al. A self-learning fuzzy discrete event system for\n",
      "HIV/AIDS treatment regimen selection. IEEE Trans Syst MAN Cybern B\n",
      "Cybern 2007;37. https://doi.org/10.1109/TSMCB.2007.895360 .\n",
      "[83] Gulshan V, Peng L, Coram M, Stumpe MC, Wu D, Narayanaswamy A, et al.\n",
      "Development and validation of a deep learning algorithm for detection of\n",
      "diabetic retinopathy in retinal fundus photographs. JAMA 2016;316:2402.https://doi.org/10.1001/jama.2016.17216 .\n",
      "[84] Rajkomar A, Hardt M, Howell MD, Corrado G, Chin MH. Ensuring fairness in\n",
      "machine learning to advance health equity. Ann Intern Med 2018;169:866.\n",
      "https://doi.org/10.7326/M18-1990 .\n",
      "[85] Gianfrancesco MA, Tamang S, Yazdany J, Schmajuk G. Potential biases in ma-\n",
      "chine learning algorithms using electronic health record data. JAMA Intern\n",
      "Med 2018;178:1544 e7.https://doi.org/10.1001/jamainternmed.2018.3763 .\n",
      "[86] Osakunor DNM, Sengeh DM, Mutapi F. Coinfections and comorbidities in\n",
      "African health systems: at the interface of infectious and noninfectious dis-\n",
      "eases. PLoS Negl Trop Dis 2018;12. https://doi.org/10.1371/\n",
      "journal.pntd.0006711 .\n",
      "[87] Taneja I, Reddy B, Damhorst G, Dave Zhao S, Hassan U, Price Z, et al. Combining\n",
      "biomarkers with EMR data to identify patients in different phases of sepsis. Sci\n",
      "Rep 2017;7:10800. https://doi.org/10.1038/s41598-017-09766-1 .[88] Ford E, Carroll JA, Smith HE, Scott D, Cassell JA. Extracting information from\n",
      "the text of electronic medical records to improve case detection: a system-\n",
      "atic review. J Am Med Inform Assoc 2016;23:1007 e15. https://doi.org/\n",
      "10.1093/jamia/ocv180 .\n",
      "[89] Greenhalgh T, Wherton J, Papoutsi C, Lynch J, Hughes G, A ’Court C, et al.\n",
      "Beyond adoption: a new framework for theorizing and evaluating non-adoption, abandonment, and challenges to the scale-up, spread, and sus-\n",
      "tainability of health and care technologies. J Med Internet Res 2017;19:e367.\n",
      "https://doi.org/10.2196/jmir.8775 .\n",
      "[90] Stead WW. Clinical implications and challenges of arti ﬁcial intelligence and\n",
      "deep learning. JAMA 2018;320:1107 e8. https://doi.org/10.1001/\n",
      "jama.2018.11029 .\n",
      "[91] Rawson TM, Charani E, Moore LSP, Hernandez B, Castro-S /C19anchez E, Herrero P,\n",
      "et al. Mapping the decision pathways of acute infection management in sec-\n",
      "ondary care among UK medical physicians: a qualitative study. BMC Med\n",
      "2016;14. https://doi.org/10.1186/s12916-016-0751-y0751-y .\n",
      "[92] Maddox TM, Rumsfeld JS, Payne PRO. Questions for arti ﬁcial intelligence in\n",
      "health care. JAMA 2018. https://doi.org/10.1001/jama.2018.18932 .\n",
      "[93] He J, Baxter SL, Xu J, Xu J, Zhou X, Zhang K. The practical implementation of\n",
      "artiﬁcial intelligence technologies in medicine. Nat Med 2019;25:30 e6.\n",
      "https://doi.org/10.1038/s41591-018-0307-0 .\n",
      "[94] Asan O, Smith P, Montague E. More screen time, less face time\n",
      "eimplications\n",
      "for EHR design. J Eval Clin Pract 2014;20:896 e901. https://doi.org/10.1111/\n",
      "jep.12182 .\n",
      "[95] Margalit RS, Roter D, Dunevant MA, Larson S, Reis S. Electronic medical re-\n",
      "cord use and physician epatient communication: an observational study of\n",
      "Israeli primary care encounters. Patient Educ Couns 2006;61:134 e41.\n",
      "https://doi.org/10.1016/j.pec.2005.03.004 .\n",
      "[96] Nemati S, Holder A, Razmi F, Stanley MD, Clifford GD, Buchman TG. An\n",
      "interpretable machine learning model for accurate prediction of sepsis in theICU. Crit Care Med 2018;46:547 e53. https://doi.org/10.1097/\n",
      "CCM.0000000000002936 .\n",
      "[97] Price WN, Cohen IG. Privacy in the age of medical big data. Nat Med 2019;25:\n",
      "37e43.https://doi.org/10.1038/s41591-018-0272-7 .\n",
      "[98] Calvert J, Hoffman J, Barton C, Shimabukuro D, Ries M, Chettipally U, et al.\n",
      "Cost and mortality impact of an algorithm-driven sepsis prediction system.\n",
      "J Med Econ 2017;20:646 e51. https://doi.org/10.1080/13696998.2017.\n",
      "1307203 .\n",
      "[99] Sinha M, Jupe J, Mack H, Coleman TP, Lawrence SM, Fraley SI. Emerging\n",
      "technologies for molecular diagnosis of sepsis. Clin Microbiol Rev 2018;31.\n",
      "https://doi.org/10.1128/CMR.00089-17 . e00089-17.\n",
      "[100] Bisaso KR, Anguzu GT, Karungi SA, Kiragga A, Castelnuovo B. A survey of\n",
      "machine learning applications in HIV clinical research and care. Comput Biol\n",
      "Med 2017;91:366 e71.https://doi.org/10.1016/j.compbiomed.2017.11.001 .\n",
      "[101] Zazzi M, Cozzi-Lepri A, Prosperi MCF. Computer-aided optimization of\n",
      "combined anti-retroviral therapy for HIV: new drugs, new drug targets and\n",
      "drug resistance. Curr HIV Res 2016;14:101 e9.\n",
      "[102] Al-Garadi MA, Khan MS, Varathan KD, Mujtaba G, Al-Kabsi AM. Using online\n",
      "social networks to track a pandemic: a systematic review. J Biomed Inform\n",
      "2016;62:1 e11.https://doi.org/10.1016/j.jbi.2016.05.005 .\n",
      "[103] Nourani E, Khunjush F, Durmus ¸ S. Computational approaches for prediction\n",
      "of pathogen-host protein-protein interactions. Front Microbiol 2015;6:94.\n",
      "https://doi.org/10.3389/fmicb.2015.00094 .\n",
      "[104] Zhou H, Jin J, Wong L. Progress in computational studies of host-pathogen\n",
      "interactions. J Bioinform Comput Biol 2013;11:1230001. https://doi.org/\n",
      "10.1142/S0219720012300018 .\n",
      "[105] Segal G. Identi ﬁcation of legionella effectors using bioinformatic approaches.\n",
      "Methods Mol Biol Clifton NJ 2013;954:595 e602. https://doi.org/10.1007/\n",
      "978-1-62703-161-5_37 .\n",
      "[106] Ackerman ME, Barouch DH, Alter G. Systems serology for evaluation of HIV\n",
      "vaccine trials. Immunol Rev 2017;275:262 e70. https://doi.org/10.1111/\n",
      "imr.12503 .\n",
      "[107] Ba /C21gcıU, Bray M, Caban J, Yao J, Mollura DJ. Computer-assisted detection of\n",
      "infectious lung diseases: a review. Comput Med Imaging Graph Off J Comput\n",
      "Med Imaging Soc 2012;36:72 e84.https://doi.org/10.1016/j.compmedimag.\n",
      "2011.06.002 .\n",
      "[108] Laketa V. Microscopy in infectious disease research - imaging across scales.\n",
      "J Mol Biol 2018;430:2612 e25.https://doi.org/10.1016/j.jmb.2018.06.018 .N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 595\n"
     ]
    }
   ],
   "source": [
    "pdf_file = open('1Machine learning for clinical decision support in infectious diseases.pdf', 'rb')\n",
    "pdf_reader = pypdf.PdfReader(pdf_file)\n",
    "num_pages = len(pdf_reader.pages)\n",
    "text = \"\"\n",
    "for page_num in range(num_pages):\n",
    "    page = pdf_reader.pages[page_num]\n",
    "    text += page.extract_text()\n",
    "print(text)\n",
    "pdf_file.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Narrative review\\nMachine learning for clinical decision support in infectious diseases: a\\nnarrative review of current applications\\nN. Peiffer-Smadja1,2,*, T.M. Rawson1, R. Ahmad1, A. Buchard3, P. Georgiou4,\\nF.-X. Lescure2,5, G. Birgand1, A.H. Holmes1\\n1)National Institute for Health Research Health Protection Research Unit in Healthcare Associated Infections and Antimicrobial Resistance, Imperi al College\\nLondon, London, UK\\n2)French Institute for Medical Research (Inserm), Infection Antimicrobials Modelling Evolution (IAME), UMR 1137, University Paris Diderot, Paris, France\\n3)Babylon Health, London, UK\\n4)Department of Electrical and Electronic Engineering, Imperial College, London, UK\\n5)Infectious Diseases Department, Bichat-Claude Bernard Hospital, Assistance-Publique H ^opitaux de Paris, Paris, France\\narticle info\\nArticle history:\\nReceived 27 June 2019Received in revised form29 August 2019Accepted 9 September 2019Available online 17 September 2019\\nEditor: A. Kalil\\nKeywords:\\nArtiﬁcial intelligence\\nClinical decision support systemInfectious diseasesInformation technologyMachine learningabstract\\nBackground: Machine learning (ML) is a growing ﬁeld in medicine. This narrative review describes the\\ncurrent body of literature on ML for clinical decision support in infectious diseases (ID).\\nObjectives: We aim to inform clinicians about the use of ML for diagnosis, classi ﬁcation, outcome pre-\\ndiction and antimicrobial management in ID.Sources: References for this review were identi ﬁed through searches of MEDLINE/PubMed, EMBASE,\\nGoogle Scholar, biorXiv, ACM Digital Library, arXiV and IEEE Xplore Digital Library up to July 2019.\\nContent: We found 60 unique ML-clinical decision support systems (ML-CDSS) aiming to assist ID cli-\\nnicians. Overall, 37 (62%) focused on bacterial infections, 10 (17%) on viral infections, nine (15%) ontuberculosis and four (7%) on any kind of infection. Among them, 20 (33%) addressed the diagnosis of\\ninfection, 18 (30%) the prediction, early detection or strati ﬁcation of sepsis, 13 (22%) the prediction of\\ntreatment response, four (7%) the prediction of antibiotic resistance, three (5%) the choice of antibioticregimen and two (3%) the choice of a combination antiretroviral therapy. The ML-CDSS were developed\\nfor intensive care units ( n¼24, 40%), ID consultation ( n¼15, 25%), medical or surgical wards ( n¼13,\\n20%), emergency department ( n¼4, 7%), primary care ( n¼3, 5%) and antimicrobial stewardship ( n¼1,\\n2%). Fifty-three ML-CDSS (88%) were developed using data from high-income countries and seven (12%)with data from low- and middle-income countries (LMIC). The evaluation of ML-CDSS was limited to\\nmeasures of performance (e.g. sensitivity, speci ﬁcity) for 57 ML-CDSS (95%) and included data in clinical\\npractice for three (5%).Implications: Considering comprehensive patient data from socioeconomically diverse healthcare set-\\ntings, including primary care and LMICs, may improve the ability of ML-CDSS to suggest decisions\\nadapted to various clinical contexts. Currents gaps identi ﬁed in the evaluation of ML-CDSS must also be\\naddressed in order to know the potential impact of such tools for clinicians and patients. N. Peiffer-\\nSmadja, Clin Microbiol Infect 2020;26:584\\n©2019 European Society of Clinical Microbiology and Infectious Diseases. Published by Elsevier Ltd. All\\nrights reserved.\\nIntroduction\\nArtiﬁcial intelligence (AI), initiated in 1956, is often de ﬁned as\\nthe study of ‘intelligent agents ’, devices that perceive theirenvironment and take actions that maximize the likelihood of\\nsuccessfully achieving their goals [ 1]. AI in healthcare began with\\nthe development of expert systems, based on rules captured from\\ninterviews with medical experts, then translated and programmed\\n[2]. With about 450 rules, MYCIN, the ﬁrst expert system in med-\\nicine, was developed in 1976 and aimed to suggest antibiotic reg-\\nimens for severe bacterial infections [ 3]. However, MYCIN was\\nnever actually used in practice because of the lack of system*Corresponding author. Nathan Peiffer-Smadja, National Institute for Health\\nResearch Health Protection Research Unit in Healthcare Associated Infections andAntimicrobial Resistance, Imperial College London, London, UK.\\nE-mail address: n.peiffer-smadja@ic.ac.uk (N. Peiffer-Smadja).\\nContents lists available at ScienceDirect\\nClinical Microbiology and Infection\\njournal homepage: www.clinicalmicrobiol ogyandinfection.com\\nhttps://doi.org/10.1016/j.cmi.2019.09.009\\n1198-743X/ ©2019 European Society of Clinical Microbiology and Infectious Diseases. Published by Elsevier Ltd. All rights reserved.Clinical Microbiology and Infection 26 (2020) 584 e595integration into clinical work. Another major limitation of expert\\nsystems is the large volume of rules needed to capture experts'\\nknowledge in dif ﬁcult clinical decisions.\\nMachine learning (ML) was developed to overcome the con-\\nstraints of expert systems [ 2]. In ML, engineers programme algo-\\nrithms able to de ﬁne their own rules from data. Thus, human hand-\\ncoded rules are replaced by the arti ﬁcialﬁnding of rules from data.\\nThis allows ML systems to learn from data and interpret unknown\\nsituations. Among the panel of ML techniques developed, deep\\nlearning based on arti ﬁcial neural networks is the most famous [ 4].\\nThe performance and ability of the machine to learn is driven by the\\nvolume and quality of data provided, that is why ML systems are\\nsometimes called data-intensive systems ( Fig. 1 ).\\nThe availability of healthcare data dramatically increased with\\nelectronic health record systems and the rise of connected devices\\n[5]. The optimal analysis and interpretation of this important vol-\\nume of data, called ‘big data ’, needs the computing power of\\nmodern machines. In this context, ML tools integrating and making\\nsense of huge amounts of complex data are becoming popular.\\nClinical decision support systems (CDSS) can be de ﬁned as software\\nin which the characteristics of an individual patient are used topresent patient-speci ﬁc assessments or recommendations to the\\nclinician towards a decision [ 6].Most current CDSS or computer-\\naided diagnosis or therapy are expert systems [ 7], but Machine\\nLearning-Clinical Decision Support Systems (ML-CDSS) are drawing\\nincreased interest ( Fig. 2 )[8e10].\\nML systems have been developed in many ﬁelds of medicine,\\nincluding radiology, with the interpretation of images from chest\\nX-ray or magnetic resonance imaging for diagnostic purposes\\n[11e15]. The ﬁrst FDA approval for an autonomous AI system\\ntook place in 2018 with IDx, a ML system used to detect diabetic\\nretinopathy in retinal fundus photographs [ 16]. In infectious\\ndiseases (IDs), most ML work focuses on research, drug devel-\\nopment or clinical microbiology. Systems have been developed to\\nanalyse bacterial genome and improve the prediction of resis-\\ntance [ 17,18], HIV genotype and predict susceptibility to antire-\\ntroviral drugs [ 19], patterns of epidemics for surveillance\\npurpose [ 20,21] or to discover new antibacterial drugs or vac-\\ncines [ 22e24](Table 1 ).\\nThis article focuses on ML adapted to clinicians' decision (ML-\\nCDSS) in ID. More precisely, we describe the objectives, character-istics, development and assessment of ML systems that may\\ndirectly help clinicians with their work to diagnose infectious dis-\\neases, predict severity, decide whether they should use an anti-\\nmicrobial and which antimicrobial to choose or what dosage.\\nMaterial and methods\\nSearch strategy\\nReferences for this review were identi ﬁed through searches of\\nMEDLINE/PubMed, EMBASE, Google Scholar, biorXiv, ACM Digital\\nLibrary, arXiV and IEEE Xplore Digital Library for articles by use of acombination of ML keywords ( ‘deep learning ’,‘artiﬁcial intelli-\\ngence ’,‘artiﬁcial learning ’,‘machine learning ’,‘machine intelli-\\ngence ’, ‘neural networks ’, ‘probabilistic networks ’, ‘knowledge\\nrepresentation ’,‘statistical learning ’,‘bayesian learning ’), decision-\\nmaking keywords ( ‘medical decision ’,‘decision tool ’,‘support tool ’,\\n‘clinical decision ’,‘physician decision ’,‘decision algorithm ’,‘CDSS ’,\\n‘clinical management ’,‘decision making ’), antimicrobial keywords\\n(‘antimicrobial ’,‘\\nantibiotic ’,‘anti-infective ’,‘antifungal ’,‘antipara-\\nsitic ’,‘antiviral ’) or sepsis keywords ( ‘sepsis ’,‘septic shock ’). We\\nincluded articles resulting from these searches and relevant refer-\\nences cited in those articles up to July 2019.Study selection\\nProspective and retrospective articles in English that reported\\noriginal research on ML-CDSS for ID were included. We included\\ndevelopment reports, implementation studies, clinical trials or\\nqualitative studies in primary, secondary, tertiary care including\\nintensive care and paediatrics. We excluded studies that describe\\nexpert system CDSS, as de ﬁned by the use of manually pro-\\ngrammed rules, studies with ML systems that use data not\\ncurrently available in routine clinical care (e.g. bacterial genome) or\\nwith outcomes irrelevant to clinicians' use for usual care (e.g.\\ndevelopment of a new biomarker).\\nDeﬁnitions of learning methods\\nML methods can be divided into supervised learning, unsuper-\\nvised learning and reinforcement learning [ 2]. Supervised learning\\nrefers to algorithms using labelled data as a training dataset.\\nLabelled data are data in which the outcome of interest is de ﬁned;\\nfor example, to train an algorithm for sepsis prediction, we use a\\ndataset in which patients are already de ﬁned as having sepsis or\\nnot. The algorithm will then choose the best model to predict theoutcome of interest. In unsupervised learning, data are used\\nwithout a prede ﬁned outcome of interest. Algorithms are left to\\ntheir own to ﬁnd patterns and to extract hidden structure from data\\nwithout any expert labelling. Unsupervised learning is mainly used\\nin medicine to do clustering, aiming to discover inherent grouping\\nin the data, such as similar groups of patients based on clinical data\\n[25]. Reinforcement learning involves algorithms discovering ac-\\ntions that yield the greatest rewards through trial and error. In this\\ncategory, the algorithm is programmed to consider survival or a\\nreduced hospital length of stay as a reward. A training dataset is\\nused by the algorithm to conduct multiple tests in order to develop\\nthe model that obtains the highest reward [ 2,26].\\nAnalysis of data in the selected articles\\nIn each article, we extracted the variables that were analysed by\\nthe ML-CDSS. We split these variables into structured and un-\\nstructured variables. Structured variables are variables in which the\\npossible values are prede ﬁned, e.g. for wound aspect, 0 ¼no ne-\\ncrosis, 1 ¼non-viable tissue, etc. Unstructured variables refer to\\nfree text such as a description of the wound by a clinician. We\\nfurther split structured variables into demographic data (e.g. age,\\nethnicity), medical or surgical history (e.g. comorbidities, date of\\nadmission), vitals (e.g. temperature), symptoms or physical exam-\\nination ﬁndings (e.g. cough, lung auscultation), laboratory workup\\n(e.g. creatinine), microbiology workup (e.g. antibiotic susceptibility\\ntesting), therapeutic history (e.g. type of surgery, administered\\ndrugs) or other workup (biopsy results, medical imaging). Medicalimaging can be considered as a structured variable, for example\\n‘presence of lung in ﬁltrates ’can be mapped a priori to a uniquely\\ndeﬁned input value, or as an unstructured variable when the image\\nitself is fed as input to the CDSS.\\nResults\\nGeneral characteristics of ML-CDSS\\nAmong the 126 abstracts identi ﬁed and assessed for eligibility,\\n14 were excluded because the variables were not available in\\nroutine clinical practice (e.g. bacterial genome), 29 because the\\nCDSS was an expert system and eight because the outcome of the\\nCDSS was not clinical (e.g. development of a novel biomarker).N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 585Fig. 1. Artiﬁcial intelligence for clinical infectious diseases.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 586In total, 75 papers with 60 unique ML-CDSS addressing clinical\\nID decisions were included. Thirty-one articles were found in\\nMEDLINE/PubMed,12 in free repository of electronic preprints such\\nas arXiv ( n¼8) or bioRxiv ( n¼4), ten in Google Scholar and ten in\\nIEEE Xplore Digital Library. The main characteristics of the ML-CDSS\\nare summarized in Table 2 and detailed in the Supplementary\\nmaterial .\\nOverall, 37 (62%) ML-CDSS focused on bacterial infections, 10\\n(17%) on viral infections, nine (15%) on tuberculosis and four (7%) on\\nany kind of infection. Among them, 20 (33%) addressed the diag-\\nnosis of infection, 18 (30%) the prediction, early detection or\\nstrati ﬁcation of sepsis, 13 (22%) the prediction of treatment\\nresponse, four (7%) the prediction of antibiotic resistance, three\\n(5%) the choice of antibiotic regimen and two (3%) the choice of a\\ncombination antiretroviral therapy (cART). Most of the ML-CDSS\\nwere developed with data from adult patients ( n¼54, 90%) but\\nsome targeted paediatric patients ( n¼3, 5%) [ 27e29] or neonates\\n(n¼3, 5%) [ 30e32].\\nThe vast majority of ML-CDSS ( n¼58, 97%) used supervised\\nlearning whereas two (3%) used reinforcement learning [ 33,34].\\nPrediction, early detection or strati ﬁcation of sepsis\\nEighteen ML-CDSS (30%) addressed the prediction, early detec-\\ntion or strati ﬁcation of sepsis in the Intensive Care Unit (ICU).\\nAmong these ML-CDSS, 16 exclusively analysed structured patient\\nvariables: vitals ( n¼15), laboratory data ( n¼12), basic de-\\nmographic information ( n¼7), medical history limited to main\\ncomorbidities and date of admission ( n¼7), therapeutic data\\n(n¼5) and electrocardiogram waveform ( n¼1). Two ML-CDSS\\nadded unstructured clinical data to their model, one speci ﬁcally\\nlooked for antibiotic prescription in nursing notes to predict sepsis\\n[35], but the other did not give details [ 36]. No CDSS for sepsis\\nprediction used symptoms, physical examination ﬁndings nor\\nmicrobiology data.\\nAll the 18 ML-CDSS were developed with data from high-income\\ncountries (HICs), with six ML-CDSS using the MIMIC (Multipa-\\nrameter Intelligent Monitoring in Intensive Care) dataset. Thisdataset contains openly available data from ~50 000 critical care\\npatients admitted to a Medical Center in Boston [ 37].Seventeen ML-CDSS were evaluated with measures of perfor-\\nmance such as sensitivity or speci ﬁcity and one ML-CDSS was\\nevaluated in the clinical setting. This ML-CDSS was developed by a\\ncompany that published a set of papers describing its development,\\nperformance and impact in quasi-experimental studies [ 38]. They\\nreported a relative reduction of in-hospital mortality between 30%\\nand 60% after implementation of their ML-CDSS in ICUs [ 39e41]\\nbut do not use statistical methods adapted to quasi-experimental\\ndesigns [ 42]. Indeed, they directly compare in-hospital mortality\\nbefore and after the use of the ML-CDSS, a measure that is sus-\\nceptible to biases such as the history bias, i.e. the possibility that the\\nchange in the outcome may be linked to other changes in the\\nenvironment rather than to the addition of the ML-CDSS [ 43]. They\\nalso published a randomized clinical trial in which 67 patients were\\nrandomized to a group monitored by a machine learning algorithm\\nand 75 patients to a control group [ 44]. The trial reported a relative\\nreduction of 58% of mortality in the group monitored by the ma-\\nchine learning algorithm in which patients had blood cultures\\ndrawn and antibiotics administered approximately 2.8 hours before\\nthe control group.\\nDiagnosis of infection\\nTwenty ML-CDSS (33%) focused on the diagnosis of infection, to\\nhelp clinicians decide if a patient has an infection, which infection\\n(e.g. viral or bacterial) or an alternative non-infectious diagnosis.\\nMore precisely, six ML-CDSS address the diagnosis of tuberculosis\\n(TB) in outpatient settings [ 45e50],ﬁve the diagnosis of bacterial\\ninfection in hospitalized patients [ 27,51e54], four the diagnosis of\\nsurgical site infection (SSI) [ 30,55e57], three the diagnosis of\\ninfection in emergency department [ 58e60] and two the distinc-\\ntion between bacterial and viral meningitis [ 61,62]. Among the ﬁve\\nML-CDSS to diagnose bacterial infection in hospitalized patients,\\ntwo ML-CDSS included any kind of infection [ 27,51], two focused on\\nthe prediction of positive blood cultures [ 53,54] and one on MRSA\\ninfection [ 52]. The ML-CDSS for patients hospitalized in surgical\\nwards aimed to diagnose SSIs following open abdominal surgery\\n[55,57] surgery for head or neck cancer [ 56] or following any\\nintervention on neonates [ 30].\\nAmong the 20 ML-CDSS, 18 exclusively analysed structured\\npatient variables: demographics data ( n¼15), medical history\\nFig. 2. Trends in the number of publications using selected search terms in PubMed in August 2019.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 587(n¼14), laboratory results ( n¼13), vitals ( n¼9), symptoms and\\nphysical examination ﬁndings ( n¼8), therapeutic history ( n¼7),\\nchest X-ray for TB diagnosis ( n¼2) and microbiology data ( n¼2).\\nTwo ML-CDSS added free clinical text; nursing notes, clinical\\nnarrative or chief complaint [ 51,59]. Seventeen ML-CDSS were\\ndeveloped with data from HICs and three with data from low- and\\nmiddle-income countries (LMICs), addressing the screening of TB in\\nSouth Africa [ 48] or the diagnosis of TB in Turkey [ 49] or Iran [ 63].\\nNineteen ML-CDSS were evaluated by measures of sensitivity,\\nspeci ﬁcity and receiver ROC curves. One study described a ML-CDSS\\ntrained to diagnose bacterial infection using six routinely available\\nblood parameters with data from 160 203 individuals and its use in\\na prospective observational cohort. Among 104 patients included,\\nthe ML-CDSS predicted a bacterial infection in three individuals\\nwho were not identi ﬁed by clinicians as having an infection on\\nadmission but were diagnosed later with a bacterial infection [ 58].\\nNo study assessed the clinical or microbiological impact of the use\\nof ML-CDSS in clinical settings.\\nDeterminants of treatment outcomes\\nTen ML-CDSS (17%) addressed the prediction of treatment suc-\\ncess for outpatients. Among them, ﬁve ML-CDSS predicted the\\nvirological response to HIV therapy [ 64e68] and three to HCVtherapy [ 69e71]. They all analysed therapeutic history but only\\nfour used viral genotype or medical history [ 65e67,71]. Therapeutic\\nhistory was limited to previous HIV or HCV therapies but not to\\nother drugs that the patient might have been taking. Two ML-CDSS\\npredicted treatment outcome in TB using demographics data (e.g.education level, homelessness), TB history including constitutional\\nsymptoms, TB treatment and structured data from the chest\\nradiograph (e.g. size of cavity) [ 72,73].\\nThree ML-CDSS (5%) addressed the prediction of C. dif ﬁcilecolitis\\ncomplications [ 74] or recurrence [ 75,76] in hospital settings. We\\nfound four ML-CDSS (7%) for the prediction of drug resistance: one\\npredicted the risk of developing multidrug-resistant TB in Chinesepatients with TB [ 77] one combined demographic data and medical\\nhistory for personalized prediction of baseline antibiotic resistance\\nin urinary tract infection [ 78] one analysed demographic data,\\nliving conditions and treatment history of patients with positive\\nblood cultures to predict baseline susceptibility to ampicillin, cef-\\ntriaxone and gentamicin [ 28] and one used ML to measure the\\nimpact of antibiotic exposure on the acquisition of colonization\\nwith extended-spectrum\\nb-lactamase-producing Gram-negative\\nbacteria [ 79].\\nThirteen ML-CDSS in this category were developed with patient\\ndata from HICs and two with patient data from LMICs to predict TB\\noutcomes, one in Pakistan [ 72] and the other in Eastern Europe [ 73]Table 1\\nList of reviews with a focus on machine learning in the ﬁeld of infectious diseases\\nTopic Title Year of\\npublicationReview\\ntypeDetailed outcome Settings Including CDSS\\nBacterial resistance Genome-based prediction of bacterial\\nantibiotic resistance [ 18]2018 Narrative Analysis of bacterial genome to\\npredict resistanceClinical microbiology\\nlabNo\\nMachine learning: novel bioinformatics\\napproaches for combating antimicrobialresistance [ 17]2017 Narrative Analysis of bacterial genome to\\npredict resistanceClinical microbiology\\nlabNo\\nSepsis Emerging technologies for molecular\\ndiagnosis of sepsis [ 99]2018 Narrative Microbiological diagnosis of\\nsepsisClinical microbiology\\nlabNo\\nPrediction of sepsis ICU Yes\\nHIV A survey of machine learning applications\\nin HIV clinical research and care [ 100]2017 Systematic HIV/AIDS clinical research and\\nmedical care studies that utilizemachine learning methodologyMultiple settings Yes\\nComputer-aided optimization of combined\\nanti-retroviral therapy for HIV: new drugs,\\nnew drug targets and drug resistance [ 101]2016 Narrative Analysis of HIV genotype to\\npredict drug susceptibility\\nin vitro or response to\\ncombination antiretroviraltherapy in vivoVirology lab Yes\\nInfection control Using online social networks to track a\\npandemic: A systematic review [ 102]2016 Systematic Analysis of online social\\nnetwork data to trackpandemicsHealth authorities No\\nAutomated surveillance of healthcare-\\nassociated infections: state of the art [ 21]2017 Narrative Surveillance of healthcare-\\nassociated infectionsInfection control unit No\\nIntroduction to machine learning in digital\\nhealthcare epidemiology [ 20]2018 Narrative Prediction, detection of trends\\nand patterns for surveillancepurposesInfection control unit No\\nMolecular biology Computational approaches for prediction of\\npathogen-host protein-protein interactions[103]2015 Narrative Prediction of pathogen-host\\nprotein-protein interactionResearch lab No\\nProgress in computational studies of host-\\npathogen interactions [ 104]2013 Narrative Prediction of pathogen-host\\nprotein-protein interactionResearch lab No\\nIdenti ﬁcation of legionella effectors using\\nbioinformatic approaches [ 105]2012 Narrative Identi ﬁcation\\nofL. pneumophila effectors\\nResearch lab No\\nVaccine development Comparative pathogenesis and systems\\nbiology for biodefense virus vaccinedevelopment [ 24]2010 Narrative Analysis of Virus-Host\\nInteractionsResearch lab/Research\\nand developmentNo\\nSystems serology for evaluation of HIV\\nvaccine trials [ 106]2017 Narrative De ﬁning humoral signatures in\\nresponse to vaccinesResearch lab/Research\\nand developmentNo\\nDrug discovery Machine-learning techniques applied to\\nantibacterial drug discovery [ 23]2014 Narrative Antibiotic drug discovery Research lab/Research\\nand developmentNo\\nMedical imaging Computer-assisted detection of infectious\\nlung diseases: a review [ 107]2013 Narrative Analysis of medical imaging of\\nrespiratory tract infectionsRadiology department No\\nMicroscopy Microscopy in infectious disease research-\\nimaging across scales [ 108]2018 Narrative Analysis of microscopic images\\nin ID researchResearch lab NoN. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 588In addition, one ML-CDSS used data both from HICs and middle-\\nincome countries [ 79]. Another ML-CDSS was developed in HICs\\nand then speci ﬁcally adapted to the speci ﬁcities of human immu-\\nnode ﬁciency virus (HIV) care in LMICs such as limited access to\\ngenotype data, infrequent visits to clinics and restricted list of\\navailable drugs [ 67]. It used the HIV Drug Resistance Database that\\nincludes large datasets from patients around the world including\\nsub-Saharan countries.\\nThe 17 ML-CDSS predicting treatment outcomes or antibiotic\\nsusceptibilities were evaluated by measures of sensitivity, speci-\\nﬁcity and receiver operating characteristic (ROC) curves.\\nTreatment selection\\nFive ML-CDSS (8%) addressed the choice of antimicrobials: three\\nML-CDSS aimed to ﬁnd the optimal antibiotic regimen [ 33,80,81]and two the optimal cART for HIV [ 34,82]. All the CDSS focused on\\nthe choice of an agent but did not give individual advice on the dose\\nor duration.\\nThe two ML-CDSS to guide cART focused on outpatient settings:\\none analysed treatment history and treatment objectives but did\\nnot consider clinical or biological data [ 82], whereas the other used\\ndemographics, medical history, CD4 þcell count, viral load, geno-\\ntypic data and treatment history [ 34]. Among the three ML-CDSS\\nfor antibiotic management, one was adapted to ICUs [ 33]. another\\none to primary care [ 81] and the third one targeted antimicrobial\\nstewardship teams [ 80]. Their use of patient data was heteroge-\\nneous; only one ML-CDSS took into account the identi ﬁcation of the\\npathogen in blood culture results but not the antimicrobial sus-\\nceptibility testing [ 33], medical history was not used in two ML-\\nCDSS [ 33,80] and limited to breastfeeding, pregnancy, allergy and\\nkidney failure in the third [ 81]. They did not use the therapeutic\\nhistory of patients, neither data from the physical examination nor\\nlocal antibiotic resistance rates and were not adapted for a dynamic\\nuse over time.\\nAll of these ML-CDSS concerned HICs, using data from North\\nAmerican or European patients and only one ML-CDSS was evalu-ated in a clinical setting. It was a ML-CDSS focusing on inappro-\\npriate prescriptions for review by antimicrobial stewardship\\npharmacists. The addition of a ML module to an expert system was\\nshown to identify inappropriate prescriptions of piperacillin-\\ntazobactam that were missed by the expert system [ 80]. Howev-\\ner, this study does not report the clinical or microbiological out-\\ncomes of this intervention.\\nDiscussionHealthcare settings\\nMost ML-CDSS found in the literature were developed for sec-\\nondary and tertiary care ( n¼57, 95%) whereas only three ML-CDSS\\n(5%) targeted primary care [ 48,78,81]. Four ML-CDSS were specif-\\nically tailored for emergency care [ 46,58e60], compared with more\\nthan 20 ML-CDSS for ICUs. Moreover, only seven ML-CDSS (12%)\\nwere adapted to LMICs [ 48,49,63,67,72,73,79], six of them con-\\ncerning HIV or TB.\\nML techniques usually need a large amount of data (e.g. more\\nthan 128 000 retinal images in a recent ophthalmologic ML sys-\\ntem) [ 83]. Thus, it is no surprise that most ML-CDSS in ID currently\\nshould focus on domains generating high-quality databases\\nincluding ICU and HIV patients from HICs. The MIMIC dataset was\\na recurrent data source for ML systems predicting sepsis [ 37]. In\\nthe same way, ML-CDSS targeting HIV therapy selection or the\\nprediction of virological response were also based on large open-\\naccess databases [ 66]. The development of ML-CDSS is more\\ndifﬁcult in primary care where available patient data for the\\nclinician is limited and databases are scarce. The same statement\\ncan be made for LMICs where data extraction is dif ﬁcult due to the\\nfrequent lack of clinical information system. The availability of\\ndata is also correlated with healthcare access: vulnerable pop-ulations tend to have a lower access to the healthcare system with\\na more fragmented care. This could undermine the ability of ML-\\nCDSS to make adequate predictions for patients from vulnerable\\npopulations and lead to a potential increase in healthcare in-\\nequalities [ 84,85].\\nThe unequal spreading of ML-CDSS across healthcare settings\\nreﬂects the unequal availability of high-quality and large clinical\\ndatabases. This is why a special effort should be made to target the\\ndevelopment of such databases in underfunded settings such as\\nprimary care or LMICs and to include data from diverse settings and\\npopulations in training and validation datasets. Global open accessTable 2\\nSummary of machine learning clinical decision support systems\\nML-CDSS characteristics n¼60 (%)\\nMedical settingsa\\nIntensive Care Unit 24 (40)\\nInfectious diseases consultation 15 (25)Medical ward 9 (15)Surgical ward 5 (8)Emergency department 4 (7)\\nPrimary care 3 (5)\\nAntimicrobial stewardship team 1 (2)\\nGeographical settings\\na\\nHigh-income countries 54 (90)Low- and middle-income countries 7 (12)\\nPopulation\\nAdults 53 (88)Neonates 3 (5)\\nPaediatric patients 3 (5)\\nRetirement-home 1 (2)\\nTypes of decision support\\nDiagnostic\\nDiagnosis of infection 20 (33)Prediction of sepsis 18 (30)Prediction of antibiotic resistance 4 (7)\\nTherapeutic\\nPrediction of treatment response 13 (22)\\nAntibiotic selection 3 (5)HIV therapy selection 2 (3)\\nType of infection\\nBacterial infection 37 (62)Viral infection 10 (17)Mycobacterial infection 9 (15)Any kind of infection 4 (7)\\nTypes of learning\\nSupervised learning 58 (97)Reinforcement learning 2 (3)\\nData\\nClinical data (e.g. demographics, vitals) 52 (87)Laboratory 38 (63)Therapy 28 (47)\\nMicrobiology 15 (25)\\nOther workup (e.g. ECG, imaging) 6 (10)Unstructured clinical data (free text) 5 (8)\\nEvaluation\\nb\\nPerformance 57 (95)Use 3 (5)Adoption 0\\nML, machine learning; CDSS, clinical decision support systems; HIV, human im-\\nmunode ﬁciency virus.\\naOne article included patients in medical wards, surgical wards and intensive\\ncare units in high and middle-income countries.\\nbWe separated studies that describe the performance (e.g. receiver operating\\ncharacteristic curves) of the ML-CDSS, studies that describe the use of the CDSS inreal-life settings and studies that describe the adoption of the CDSS in routineclinical practice.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 589to these databases should be a requirement for future funding in\\norder to increase the opportunities for ML-CDSS development\\neverywhere in the world. Moreover, we have not found any study\\nfocusing on fungal or parasitic diseases in this review. These areas\\nmay represent interesting opportunities for developing CDSS using\\nmachine learning for diagnosis, prediction of resistance or choice of\\nantifungal or antiparasitic therapy. The analysis of the outputs of\\nML-CDSS at the population level should also be encouraged as it\\nmay be useful for public health and research institutes to regularly\\nmonitor trends in diagnosis and prescribing and correct potential\\nbiases.\\nIf these constraints and potential issues are considered, ML-\\nCDSS appear as credible options for supporting infection manage-\\nment in countries where ID specialists and antimicrobial teams\\nmay not be readily available [ 86].\\nChoice of patient variables\\nIn the present review, many ML-CDSS did not include some\\nclinically relevant data. Medical history analysed in ML-CDSS was\\noften limited to the presence of two or three prede ﬁned diseases\\n(kidney failure, cirrhosis, etc.) [ 56,87]. No ML-CDSS for the diag-\\nnosis of infection considered patients exposed to infected in-\\ndividuals or high-risk environment (e.g. travel history). We did not\\nﬁnd any ML-CDSS predicting sepsis and taking into account the\\nresults from blood cultures or from any other microbiology sample.\\nAmong the 60 ML-CDSS, chest X-Ray was the only medical imaging\\nanalysed [ 46,48,72]. No ML-CDSS to predict antibiotic resistance or\\nto select an antibiotic regimen included data from local antibiotic\\nresistance rates. Paradoxically, ML-CDSS often use less data than a\\nhuman clinician would do. In some articles, the authors analysed\\nthe performance of ML-CDSS when using a reduced set of variables\\nand they found that the sensitivity and speci ﬁcity of ML-CDSS were\\nsystematically better when they used a larger panel of variables\\n[60,87,48]. especially when adding unstructured data [ 59,36].\\nEnsuring integration of physical examination or detailed medical\\nhistory in these systems is of critical importance. Progress in Nat-\\nural Language Processing (NLP, i.e. the ability of computers to\\nanalyse human language) may help the integration of free medicaltext in future decision systems [ 88]. Nonetheless, the development\\nof ML-CDSS using minimal variables may be of interest when data\\nare not available across some areas or in resource limited settings. A\\nparticular attention should be paid to which variables are used by\\nthe ML-CDSS to predict their outcome: e.g. we found a ML-CDSS\\nthat used the prescription of antibiotics in ICU to predict sepsis\\n[35], which could provide good performance but seems clinically\\nirrelevant.\\nML-CDSS are constrained by the quality and availability of the\\nclinical data used for their development and validation. To ensure\\nthat future ML tools are useful for clinicians, efforts should be made\\nto provide comprehensive databases that include relevant clinical\\ndata.\\nEvaluation and use of ML-CDSS\\nThe evaluation of ML-CDSS in clinical ID is still lacking with 57\\nML-CDSS (95%) reporting measures of performance such as sensi-\\ntivity and speci ﬁcity and only three (5%) evaluated in clinical\\npractice. The validation of the performance in the original database\\nshould preferably be done in a prospective database collected for\\nvalidation purpose in another location. Then, pilot studies testing\\nthe usability and usefulness of ML-CDSS in clinical settings should\\nbe conducted, followed by clinical trials, preferably randomized,\\nevaluating patients' outcomes, process improvement or cost-\\neffectiveness.\\nSpecial attention should be paid to the integration and\\nimplementation of systems into clinical practice, and their adop-\\ntion and utilization by clinicians [ 89]. Co-design including clini-\\ncians, pharmacists engineers, data managers and service users\\nmay be key for success and may allow to study the interaction\\nbetween CDSS and healthcare professionals [ 90]. The place of ML-\\nCDSS in the work ﬂow of clinicians has to be considered as\\ndifferent clinical tasks are likely to require different degrees of\\nhuman involvement ( Fig. 3 )[10]. Data should be easily entered\\ninto the CDSS, ideally by automatic extraction from the EHR, and\\ndaily refreshed with new patient information. The CDSS need to ﬁt\\ninto the complex medical decision-making pathway and support\\nclinicians at different stages sometimes by presenting different\\nFig. 3. Interaction between clinicians and machine-learning systems according to automation level.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 590aspects of data [ 91]. According to the numerous limits that we\\nfound concerning ML-CDSS in ID, the replacement of physicians\\nby ML tools seems highly unlikely. AI is best suited to augmenting\\nhuman intelligence, bringing big-data into focus to support hu-\\nman decision-making and developers should aim to emphasize\\nthe synergy and interaction between physicians and algorithms\\ninstead of reporting less interesting clinical-versus-algorithm\\ncomparison [ 8,92,93] Clinicians' expectations are that ML-CDSS\\nwill relieve them of repetitive tasks that need to be performed\\nwithout errors and allow them to focus on tasks that require\\nhuman skills: analysis of the broader context, abstraction capa-\\nbility but also exercising empathy and building relationships.\\nHowever, we have not found any in-depth describing of the\\ninteraction of physicians with ML-CDSS in real-world settings.\\nImplementation research may help to take into account the global\\ncontext in which ML-CDSS are used [ 7].\\nUnexpected consequences of the use of ML also have to be\\nincluded in the research agenda. Consequences commonly cited are\\nthe risk of ‘deskilling ’clinicians who use ML-CDSS routinely, the\\nrisk of a blind obedience to the CDSS or the risk of decreased\\ninteraction between the clinician and the patient. Studies haveshown that the development of the electronic health record (EHR)\\nshifted clinicians' time from the patient to the computer and ML\\nsystems may represent an additional risk to do so [ 94,95]. By\\ndeﬁnition, unexpected consequences cannot be fully anticipated\\nand research on ML-CDSS in real-world settings should aim to\\ncollect information on consequences both at the clinician and pa-\\ntient -levels.Perspectives\\nInTable 3 we summarize some essential criteria for future ML-\\nCDSS adapted for clinicians' use. Clinicians must know when and\\nwhy errors by ML in clinical-decision predictions might occur and\\nwhen they cannot rely on an automated system whose intrinsic\\nmechanisms they do not understand (the so-called ‘black box ’)[8].\\nVisualization tools highlighting the variable or combination of\\nvariables that determined the output ought to be available for the\\nprescriber and are an important parameter in future ML-CDSS\\n[28,96]. ML-CDSS should also detail the trade-off that they apply\\nbetween risk-taking and caution and show the con ﬁdence interval\\nof their decisions and suggestions [ 9]. The impact of under- and\\noverdiagnosis or treatment is not the same when considering up-\\nper respiratory tract infection or sepsis therapy. Besides system\\nimprovement, we need to encourage and fund the education of\\nclinicians about AI and ML in order to develop an AI-literate\\nworkforce in healthcare worldwide.\\nIn addition, global challenges for the development of arti ﬁcial\\nintelligence and ML for clinical decisions are to be considered.\\nTable 4 describes the main challenges for ML-CDSS that have been\\nidenti ﬁed in the literature. For example, controls have to ensure\\nthat ML-CDSS are registered on public databases, thoroughly\\nevaluated independently of economic interest and validated using\\nstandardized frameworks. The privacy protection of data [ 97], cost-\\neffectiveness research at the organizational and global level [ 89,98],\\nand ethics discussion about accountability and liability when using\\nML-CDSS are other potential challenges to deal with.\\nTable 3\\nNecessary requirements for clinical decision support systems using machine learning\\nTarget Type of decision A clinically meaningful decision should be targeted, for example the diagnosis of a bacterial infection requiring antibioti cs or the\\nchoice of an antibiotic regimen\\nJustiﬁcation of ML The use of machine learning instead of expert systems should be justi ﬁed, for example by the analysis of big data that could not\\nbe captured by expert systems.\\nsettings ML-CDSS should not only be developed for secondary or tertiary care but should also target such primary care prescribers as GPs\\nor nurse prescribersThe ML-CDSS should offer a good performance in diverse settings including low- and middle-income countries\\nPatients The ML-CDSS should be performant for different populations of patients like patients with diabetes, chronic kidney disease,\\ncirrhosis, etc.If some comorbidities prevent the use of the ML-CDSS, it should be clearly stated and explained\\nUsers Different practitioners may not need the same tools even for the same decision. For example, surgeons do not have the same\\nneed for ML-CDSS addressing antimicrobial management as antimicrobial stewardship specialists or nurse prescribers inprimary care\\nData Comprehensiveness ML-CDSS should be able to analyse data from different sources such as structured clinical data, vitals, laboratory data,\\ntherapeutic history but also unstructured clinical data (e.g. free medical text, medical imaging)\\nAvailability ML-CDSS should not require data that are not available for most patients\\nExtraction ML-CDSS should be able to automatically extract data from the electronic health record (EHR)\\nDecision Uncertainty A way to present uncertainty should be found by the developers: e.g. it could give a con ﬁdence interval for the diagnosis of\\nbacterial infection or a number of suggestions for antimicrobial therapy with associated probabilities of success according to itscalculation\\nClarity ML-CDSS should display what parameters were the most important for the suggested diagnosis or therapeutic decisionFlexibility Clinicians should be able to take out a speci ﬁc clinical or laboratory element that they do not think reliable and thus have access\\nto modi ﬁed suggestions. They should also be able to add or emphasize elements that they ﬁnd particularly important or reliable.\\nMemory Clinicians need ML-CDSS that can keep track of patients and for whom the data can be updated to modify the suggestions of the\\nsystem\\nLimits ML-CDSS should display very clearly what parameters have been used or not, e.g. medical history, medical imaging, etc. and\\nother potential limits of its decision\\nInterface Integration The ML-CDSS should ideally be integrated in the EHR used by the prescriber. A special attention should be paid to the\\ninteroperability of ML-CDSS across different EHR systems\\nSpeed The ML-CDSS should be quick to use, should automatically extract pertinent data from the EHR and give its output immediatelyUser-friendly Clinicians should be able to use ML-CDSS without speci ﬁc training\\nStandardization As multiple ML-CDSS may be available for different diseases and conditions, interfaces for ML-CDSS should be standardized to a\\ncommon format to increase usability. We should avoid having a software for every decision, e.g. diagnosis, antimicrobialprescribing, dosing, prevention of venous thromboembolic disease, etc.\\nML-CDSS, machine learning clinical decision support systems.N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 591Table 4\\nGlobal challenges for machine learning clinical decision support systems\\nChallenges Description Potential solutions\\nData quantity There is a need for a huge quantity of clinical data to develop a machine\\nlearning (ML) system, especially for unsupervised MLSharable/open-source database such as MIMIC-III, a freely accessible\\ncritical care database that has helped to develop many ML-CDSS forsepsis in intensive care units\\nData quality ML-clinical decision support systems (CDSS) developers need to have\\naccess to high-quality data for training and validation dataset, especiallyfor supervised MLElectronic health record data standardization\\nPre-processing of data should be detailed in the description of ML-CCDSS\\nComprehensiveness ML-CDSS must be able to analyse data from different sources such as\\nstructured clinical data, vitals, laboratory data but also unstructured freetext (e.g. physical examination, medical history, etc.).Developers should try to include all the sources of data that are used by\\nclinicians to take clinical decisionClinicians should be included during development to make sure thatunstructured data are not left out in ML-CDSS (co-design)\\nAvailability The data that are analysed by the system should be available for most\\npatients (e.g. a CDSS analysing genetic data could not be used in usualcare)Importance of co-design with users, physicians, other prescribers and\\nclinical managers to know what data is available for patientsPossibility of adapting the ML-CDSS to different settings in whichavailable data could be different\\nInteroperability A multitude of technology platforms are involved in the generation,\\ncollection and mobilization of health data. ML-CDSS should be able tohave good performance with different platforms and electronic healthrecord systems.Initiatives as FHIR (Fast Healthcare Interoperability Resources) that\\nstandardize healthcare data to common format\\nEquity Most ML-CDSS are data-driven. As patients with low socioeconomic\\nstatus usually have a limited access to healthcare, less data from thesepatients are available and ML systems could create new inequalities orincrease health disparities. Patients with comorbidities should also be\\nincluded in ML-CDSS.Developing ML-CDSS with training and validation datasets from\\nsocioeconomically diverse health care systemsEnsuring that patients with comorbidities (e.g. chronic kidney disease,cirrhosis) are included in the training dataset\\nEnsuring that social determinants of health are being considered in the\\nsupport systemTest ML-CDSS for potential discriminatory behaviour\\nAdaptability ML-CDSS should be usable in different clinical settings including\\nprimary, secondary and tertiary care. A special attention should be paidto low- and middle-income countries that should not be left out byprogress in the development of AI.Diverse datasets coming from different countries\\nSpeci ﬁcities of LMICs should be considered early in the development of\\nML-CDSSAvailability of antimicrobials in every healthcare setting should beconsidered\\nInterpretability Clinicians should be able to understand the output of ML-CDSS and their\\ndeterminants for an intelligent use and interpretationEducation of an AI-literate clinician workforce (e.g. training in\\ninformatics and data science by the National Library of Medicine)Highlighting the variable or combination of variables that determinedthe output (e.g. ‘saliency map ’)\\nReporting of con ﬁdence level in the prediction\\nSecurity Alternatives should be developed to make sure that healthcare systems\\ncould function normally in case of an electronic problem. ML-CDSS\\nshould also be protected against hacking and other ill-willed practicesSystematic debugging\\nSecurity software\\nAnticipation and preparedness\\nValidation ML-CDSS should be validated by an independent organization. Both the\\nrisk of under-regulation and over-regulation should be consideredQuality standards, precerti ﬁcation and certi ﬁcation\\nSuch regulatory framework as the Software as Medical Device (SaMD)framework developed by the International Medical Device RegulatorsForum (IMDRF) working group chaired by the Food and DrugAdministration (FDA)\\nConﬁdence ML-CDSS developers should pay special attention to the con ﬁdence of\\nclinicians and patients in arti ﬁcial intelligence. Limited con ﬁdence or\\nfear of the users would limit the use of ML and blind con ﬁdence also\\ncalled automation bias could be dangerousBehavioural science, sociological and anthropological research on\\nartiﬁcial intelligence in medicine\\nTraining and education of clinicians about the bene ﬁts and limits of\\nartiﬁcial intelligence and machine learning\\nStudies on the interaction between machines and speci ﬁcally human\\nskillsCo-design between engineers and clinicians\\nEvaluation Evidence of the utility and bene ﬁts of ML-CDSS should not be limited to\\nanalysis of performance (e.g. ROC curves) but should consider clinicaloutcomes. Ideally, ML-CDSS should be evaluated in real-world settings\\nfrom pilot study to clinical impact in real-world settingsStudies should not limit their outcomes to the evaluation of\\nperformance but should include clinical and microbiological outcomesRandomized controlled trial should try to determine the improvement\\nof medical decision with the addition of ML-CDSS instead of comparing\\nsupport systems with human clinicians\\nAccountability The accountability of each actor (clinicians, ML-CDSS developers,\\ncerti\\nﬁcation organisms) should be de ﬁned beforehandResearch and forums on the accountability of AI and ML systems\\nInspiration from the work that has been done in other ﬁelds as\\ndriverless cars\\nCost Cost of ML-CDSS including development, data processing,\\nimplementation and impact on healthcare should be systematically\\nstudied, at the organizational and global levelCost-effectiveness analysis and research\\nPrivacy ML-CDSS should respect the privacy of health data and ensure proper\\nconsent and patient governance in data collectionGlobal regulation as the European Union General Data Protection\\nRegulation or United States Health Insurance Portability andAccountability ActUse of anonymized or pseudonymized dataPrivacy audits\\nIndependence Regulation agencies should make sure that ML-CDSS are independent\\nfrom drug and diagnostic test companies and do not favour any\\ntreatments or tests for economic purposesRegulatory frameworks should consider the need for independence\\nPublic funding for the development of AI and ML in healthcare\\nUnexpected\\nconsequencesML-CDSS have the potential to deeply modify health systems and\\nunexpected consequences should be systematically considered andstudiedSystematic research on the limits of the ML-CDSS\\nStudying potentially unexpected consequences on clinicians like‘deskilling ’(loss of clinicians' skill due to the repeated use of ML-CDSS)\\nor burn-outN. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 592Conclusion\\nCurrent ML tools cover a range of clinical outcomes such as the\\nprediction of sepsis in ICUs, the diagnosis of TB or SSI, the predic-\\ntion of virological success of cART or the selection of an antibiotic\\nregimen. However, 57 ML-CDSS out of 60 only reported perfor-\\nmance measures such as sensitivity or speci ﬁcity and evidence is\\nlacking regarding their use and impact in real-life clinical settings.\\nMoreover, current ML-CDSS mainly use patient data from HICs,relying on large available open-access datasets and generally\\nanalyse a limited number of structured patient variables. Future\\nML-CDSS in ID should be developed in diverse health settings,\\nincluding primary care and LMICs that are currently underrepre-\\nsented, and be embedded in a structured process of integration into\\nclinical settings. Future studies should aim to report clinical out-\\ncomes following sustainable use in routine clinical care.\\nTransparency Declaration\\nDr Buchard reports that he is employed by Babylon Health. Dr\\nBuchard has provided an expertise on machine learning methods\\nexclusively on selected articles sent by the corresponding author.\\nNo tools developed by Babylon Health are discussed in the review.\\nThe authors have no other relevant con ﬂicts of interest to\\ndisclose.\\nFunding\\nThe research was funded by a PhD studentship from ^Ile-de-\\nFrance Regional Health Agency, Paris, France to Nathan Peiffer-\\nSmadja, by a grant from the French Association de Chi-\\nmioth /C19erapie Anti-Infectieuse (ACAI) to Nathan Peiffer-Smadja and\\nby the National Institute for Health Research Health Protection\\nResearch Unit (NIHR HPRU) in Healthcare Associated Infection and\\nAntimicrobial Resistance at Imperial College London in partnership\\nwith Public Health England (PHE). The views expressed are those of\\nthe author(s) and not necessarily those of the NHS, the NIHR, the\\nDepartment of Health or Public Health England.\\nAcknowledgements\\nThe support of the Economic and Social Research Council (ESRC)\\nas part of the Antimicrobial Cross Council initiative supported by\\nthe seven UK research councils, and also the support of the Global\\nChallenges Research Fund, are gratefully acknowledged.\\nAppendix A. Supplementary data\\nSupplementary data to this article can be found online at\\nhttps://doi.org/10.1016/j.cmi.2019.09.009 .References\\n[1]Crevier D. A.I: The tumultuous history of the search for arti ﬁcial intelligence.\\nNew York, NY, USA: Basic Books, Inc.; 1993 .\\n[2]Mitchell TM. Machine learning. 1st ed. New York, NY, USA: McGraw-Hill,\\nInc.; 1997 .\\n[3] Shortliffe EH. Computer-based medical consultations: mycin. Elsevier; 1976.\\nhttps://doi.org/10.1016/B978-0-444-00179-5.50008-1 .\\n[4] Hinton G. Deep learning - a technology with the potential to transform health\\ncare. JAMA 2018;320:1101 e2.https://doi.org/10.1001/jama.2018.11100 .\\n[5] Beam AL, Kohane IS. Big data and machine learning in health care. JAMA\\n2018;319:1317 e8.https://doi.org/10.1001/jama.2017.18391 .\\n[6]Sim I, Gorman P, Greenes RA, Haynes RB, Kaplan B, Lehmann H, et al. Clinical\\ndecision support systems for the practice of evidence-based medicine. J Am\\nMed Inform Assoc JAMIA 2001;8:527 e34.\\n[7] Rawson TM, Moore LSP, Hernandez B, Charani E, Castro-Sanchez E, Herrero P,\\net al. A systematic review of clinical decision support systems for antimi-\\ncrobial management: are we failing to investigate these interventions\\nappropriately? Clin Microbiol Infect 2017;23:524 e32. https://doi.org/\\n10.1016/j.cmi.2017.02.028 .\\n[8] Cabitza F, Rasoini R, Gensini GF. Unintended consequences of machine\\nlearning in medicine. JAMA 2017;318:517. https://doi.org/10.1001/\\njama.2017.7797 .\\n[9] Challen R, Denny J, Pitt M, Gompels L, Edwards T, Tsaneva-Atanasova K.\\nArtiﬁcial intelligence, bias and clinical safety. BMJ Qual Saf 2019. https://\\ndoi.org/10.1136/bmjqs-2018-008370 . bmjqs-2018-008370.\\n[10] Rawson TM, Ahmad R, Toumazou C, Georgiou P, Holmes AH. Arti ﬁcial in-\\ntelligence can improve decision-making in infection management. Nat HumBehav 2019;3:543 e5.https://doi.org/10.1038/s41562-019-0583-9 .\\n[11] Choy G, Khalilzadeh O, Michalski M, Do S, Samir AE, Pianykh OS, et al. Cur-\\nrent applications and future impact of machine learning in radiology. Radi-\\nology 2018;288:318 e28.https://doi.org/10.1148/radiol.2018171820 .\\n[12] Lindsey R, Daluiski A, Chopra S, Lachapelle A, Mozer M, Sicular S, et al. Deep\\nneural network improves fracture detection by clinicians. Proc Natl Acad Sci\\n2018;115:11591 e6.https://doi.org/10.1073/pnas.1806905115 .\\n[13] Jha S, Topol EJ. Adapting to arti ﬁcial intelligence: radiologists and patholo-\\ngists as information specialists. JAMA 2016;316:2353. https://doi.org/\\n10.1001/jama.2016.17438 .\\n[14] Esteva A, Kuprel B, Novoa RA, Ko J, Swetter SM, Blau HM, et al. Dermatolo-\\ngist-level classi ﬁcation of skin cancer with deep neural networks. Nature\\n2017;542:115 e8.https://doi.org/10.1038/nature21056 .\\n[15] Topol EJ. High-performance medicine: the convergence of human and arti-\\nﬁcial intelligence. Nat Med 2019;25:44 e56.https://doi.org/10.1038/s41591-\\n018-0300-7 .\\n[16] Abr /C18\\namoff MD, Lavin PT, Birch M, Shah N, Folk JC. Pivotal trial of an auton-\\nomous AI-based diagnostic system for detection of diabetic retinopathy in\\nprimary care of ﬁces. Npj Digit Med 2018;1:39. https://doi.org/10.1038/\\ns41746-018-0040-6 .\\n[17] Macesic N, Polubriaginof F, Tatonetti NP. Machine learning: novel bioinfor-\\nmatics approaches for combating antimicrobial resistance. Curr Opin Infect\\nDis 2017;30:511 e7.https://doi.org/10.1097/QCO.0000000000000406 .\\n[18] Su M, Satola SW, Read TD. Genome-based prediction of bacterial antibiotic\\nresistance. J Clin Microbiol 2018. https://doi.org/10.1128/JCM.01405-18 .\\n[19] Zazzi M, Cozzi-Lepri A, Prosperi MCF. Computer-aided optimization of\\ncombined anti-retroviral therapy for HIV: new drugs, new drug targets and\\ndrug resistance. Curr HIV Res 2016;14:101 e9.https://doi.org/10.2174/\\n1570162X13666151029102254 .\\n[20] Roth JA, Battegay M, Juchler F, Vogt JE, Widmer AF. Introduction to machine\\nlearning in digital healthcare epidemiology. Infect Control Hosp Epidemiol\\n2018;39:1457 e62.https://doi.org/10.1017/ice.2018.265 .\\n[21] Sips ME, Bonten MJM, van Mourik MSM. Automated surveillance of\\nhealthcare-associated infections: state of the art. Curr Opin Infect Dis\\n2017;30:425 e31.https://doi.org/10.1097/QCO.0000000000000376 .\\n[22] Lee EY, Lee MW, Fulan BM, Ferguson AL, Wong GCL. What can machine\\nlearning do for antimicrobial peptides, and what can antimicrobial peptides\\ndo for machine learning? Interface Focus 2017;7:20160153. https://doi.org/\\n10.1098/rsfs.2016.0153 .\\n[23] Durrant JD, Amaro RE. Machine-learning techniques applied to antibacterial\\ndrug discovery. Chem Biol Drug Des 2015;85:14 e21. https://doi.org/\\n10.1111/cbdd.12423 .Table 4 (continued )\\nChallenges Description Potential solutions\\nImplementation The implementation of ML-CDSS should be thought of and organized as\\nearly as possible in the developmentML-CDSS developing teams should include specialists from\\nimplementation scienceEvaluation should include research on implementation, adoption, useand sustainabilitySpeci ﬁc task force committees to deal with AI implementation issues\\nmay be useful for developing a common vision at a specialty-wide level\\nSustainability ML-CDSS should be sustainable and updated according to new research,\\ndata or regulationSpeci ﬁc and sustainable funding should be given to developers and\\nadopters of ML-CDSSN. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 593[24] Bowick GC, Barrett ADT. Comparative pathogenesis and systems biology for\\nbiodefense virus vaccine development. J Biomed Biotechnol 2010;2010:\\n236528. https://doi.org/10.1155/2010/236528 .\\n[25] Deo RC. Machine learning in medicine. Circulation 2015;132:1920 e30.\\nhttps://doi.org/10.1161/CIRCULATIONAHA.115.001593 .\\n[26] Russell S, Norvig P. Arti ﬁcial intelligence: a modern approach. Third. Pear-\\nson; 2016 .\\n[27] Lamping F, Jack T, Rübsamen N, Sasse M, Beerbaum P, Mikolajczyk RT, et al.\\nDevelopment and validation of a diagnostic model for early differentiation of\\nsepsis and non-infectious SIRS in critically ill children - a data-driven\\napproach using machine-learning algorithms. BMC Pediatr 2018;18:112.\\nhttps://doi.org/10.1186/s12887-018-1082-2 .\\n[28] Oonsivilai M, Mo Y, Luangasanatip N, Lubell Y, Miliya T, Tan P, et al. Using\\nmachine learning to guide targeted and locally-tailored empiric antibiotic\\nprescribing in a children ’s hospital in Cambodia. Wellcome Open Res 2018;3.\\nhttps://doi.org/10.12688/wellcomeopenres.14847.1 .\\n[29] Desautels T, Hoffman J, Barton C, Mao Q, Jay M, Calvert J, et al. Pediatric\\nsevere sepsis prediction using machine learning. bioRxiv 2017:223289.https://doi.org/10.1101/223289 .\\n[30] Bartz-Kurycki MA, Green C, Anderson KT, Alder AC, Bucher BT, Cina RA, et al.\\nEnhanced neonatal surgical site infection prediction model utilizing statis-\\ntically and clinically signi ﬁcant variables in combination with a machine\\nlearning algorithm. Am J Surg 2018;216:764 e77.https://doi.org/10.1016/\\nj.amjsurg.2018.07.041 .\\n[31] Hu Y, Lee VCS, Tan K. Prediction of clinicians ’treatment in preterm infants\\nwith suspected late-onset sepsis - an ML approach. In: 2018 13th IEEE\\nconference on industrial electronics and applications (ICIEA); 2018.\\np. 1177 e82.https://doi.org/10.1109/ICIEA.2018.8397888 .\\n[32] Mani S, Ozdas A, Aliferis C, Varol HA, Chen Q, Carnevale R, et al. Medical\\ndecision support using machine learning for early detection of late-onsetneonatal sepsis. J Am Med Inform Assoc JAMIA 2014;21:326 e36.https://\\ndoi.org/10.1136/amiajnl-2013-001854 .\\n[33] Tsoukalas A, Albertson T, Tagkopoulos I. From data to optimal decision\\nmaking: a data-driven, probabilistic machine learning approach to decision\\nsupport for patients with sepsis. JMIR Med Inform 2015;3:e11. https://\\ndoi.org/10.2196/medinform.3445 .\\n[34] Parbhoo S, Bogojeska J, Zazzi M, Roth V, Doshi-Velez F. Combining kernel and\\nmodel based learning for HIV therapy selection. AMIA Summits Transl Sci\\nProc 2017;2017:239 e48.\\n[35] Apostolova E, Velez T. Toward automated early sepsis alerting: identifying\\ninfection patients from nursing notes. ArXiv180903995 Cs. 2018. http://\\narxiv.org/abs/1809.03995 . [Accessed 20 February 2019].\\n[36] Culliton P, Levinson M, Ehresman A, Wherry J, Steingrub JS, Gallant SI. Pre-\\ndicting severe sepsis using text from the electronic health record.\\nArXiv171111536 Cs. 2017. http://arxiv.org/abs/1711.11536 . [Accessed 30\\nJanuary 2019].\\n[37] Johnson AEW, Pollard TJ, Shen L, Lehman LH, Feng M, Ghassemi M, et al.\\nMIMIC-III, a freely accessible critical care database. Sci Data 2016;3:160035.\\nhttps://doi.org/10.1038/sdata.2016.35 .\\n[38] Calvert JS, Price DA, Chettipally UK, Barton CW, Feldman MD, Hoffman JL,\\net al. A computational approach to early sepsis detection. Comput Biol Med\\n2016;74:69 e73.https://doi.org/10.1016/j.compbiomed.2016.05.003 .\\n[39] McCoy A, Das R. Reducing patient mortality, length of stay and readmissions\\nthrough machine learning-based sepsis prediction in the emergencydepartment, intensive care unit and hospital ﬂoor units. BMJ Open Qual\\n2017;6:e000158. https://doi.org/10.1136/bmjoq-2017-000158\\n.\\n[40] Burdick H, Pino E, Gabel-Comeau D, Gu C, Huang H, Lynn-Palevsky A, et al.\\nEvaluating a sepsis prediction machine learning algorithm in the emergency\\ndepartment and intensive care unit: a before and after comparative study.bioRxiv 2018. https://doi.org/10.1101/224014 .\\n[41] Burdick H, Pino E, Gabel-Comeau D, McCoy A, Gu C, Roberts J, et al. Effect of a\\nsepsis prediction algorithm on patient mortality, length of stay, and read-\\nmission. bioRxiv 2018. https://doi.org/10.1101/457465 .\\n[42] B €arnighausen T, Oldenburg C, Tugwell P, Bommer C, Ebert C, Barreto M, et al.\\nQuasi-experimental study designs series-paper 7: assessing the assumptions.\\nJ Clin Epidemiol 2017;89:53 e66. https://doi.org/10.1016/j.jclinepi.2017.\\n02.017 .\\n[43] Ho AMH, Phelan R, Mizubuti GB, Murdoch JAC, Wickett S, Ho AK, et al. Bias in\\nbefore-after studies: narrative overview for anesthesiologists. Anesth Analg\\n2018;126:1755 e62.https://doi.org/10.1213/ANE.0000000000002705 .\\n[44] Shimabukuro DW, Barton CW, Feldman MD, Mataraso SJ, Das R. Effect of a\\nmachine learning-based severe sepsis prediction algorithm on patient sur-\\nvival and hospital length of stay: a randomised clinical trial. BMJ Open Respir\\nRes 2017;4:e000234. https://doi.org/10.1136/bmjresp-2017-000234 .\\n[45] Saybani MR, Shamshirband S, Golzari Hormozi S, Wah TY, Aghabozorgi S,\\nPourhoseingholi MA, et al. Diagnosing tuberculosis with a novel supportvector machine-based arti ﬁcial immune recognition system. Iran Red Cres-\\ncent Med J 2015;17. https://doi.org/10.5812/ircmj.17(4)2015.24557 .\\n[46] El-Solh AA, Hsiao C-B, Goodnough S, Serghani J, Grant BJB. Predicting active\\npulmonary tuberculosis using an arti ﬁcial neural network. Chest 1999;116:\\n968e73.https://doi.org/10.1378/chest.116.4.968 .\\n[47] Ansari AQ, Gupta NK, Ekata. Adaptive neurofuzzy system for tuberculosis. In:\\n2012 2nd IEEE international conference on parallel, distributed and gridcomputing; 2012. p. 568 e73.https://doi.org/10.1109/PDGC.2012.6449883 .[48] Melendez J, S /C19anchez CI, Philipsen RHHM, Maduskar P, Dawson R, Theron G,\\net al. An automated tuberculosis screening strategy combining X-ray-based\\ncomputer-aided detection and clinical information. Sci Rep 2016;6. https://\\ndoi.org/10.1038/srep25265 .\\n[49] Er O, Temurtas F, Tanr ıkulu AÇ. Tuberculosis disease diagnosis using arti ﬁcial\\nneural networks. J Med Syst 2010;34:299 e302. https://doi.org/10.1007/\\ns10916-008-9241-x .\\n[50] Santos AM, Pereira BB, Seixas JM, Mello FCQ, Kritski AL. Neural networks: an\\napplication for predicting smear negative pulmonary tuberculosis. In:\\nAuget J-L, Balakrishnan N, Mesbah M, Molenberghs G, editors. Advances in\\nstatistical methods for the health sciences. Boston, MA: Birkh €auser Boston;\\n2007. p. 275 e87.https://doi.org/10.1007/978-0-8176-4542-7_18 .\\n[51] Revuelta-Zamorano P, S /C19anchez A, Rojo-/C19Alvarez JL,/C19Alvarez-Rodríguez J,\\nRamos-L /C19opez J, Soguero-Ruiz C. Prediction of healthcare associated in-\\nfections in an intensive care unit using machine learning and big data tools.\\nIn: Kyriacou E, Christo ﬁdes S, Pattichis CS, editors. XIV mediterranean con-\\nference on medical and biological engineering and computing 2016, vol. 57.\\nCham: Springer International Publishing; 2016. p. 840 e5.https://doi.org/\\n10.1007/978-3-319-32703-7_163 .\\n[52] Hartvigsen T, Sen C, Brownell S, Teeple E, Kong X, Rundensteiner E. Early\\nprediction of MRSA infections using electronic health records. In: Pro-\\nceedings of the 11th international joint conference on biomedical engi-\\nneering systems and technologies; 2018. p. 156 e67.https://doi.org/10.5220/\\n0006599601560167 .\\n[53] Hernandez B, Herrero P, Rawson TM, Moore LSP, Evans B, Toumazou C,\\net al. Supervised learning for infection risk inference using pathology data.\\nBMC Med Inform Decis Mak 2017;17. https://doi.org/10.1186/s12911-017-\\n0550-1 .\\n[54] Van Steenkiste T, Ruyssinck J, De Baets L, Decruyenaere J, De Turck F,\\nOngenae F, et al. Accurate prediction of blood culture outcome in theintensive care unit using long short-term memory neural networks. Artif\\nIntell Med 2018. https://doi.org/10.1016/j.artmed.2018.10.008 .\\n[55] Sanger PC, van Ramshorst GH, Mercan E, Huang S, Hartzler AL,\\nArmstrong CAL, et al. A prognostic model of surgical site infection using daily\\nclinical wound assessment. J Am Coll Surg 2016;223:259 e70.https://doi.org/\\n10.1016/j.jamcollsurg.2016.04.046 . e2.\\n[56] Kuo P-J, Wu S-C, Chien P-C, Chang S-S, Rau C-S, Tai H-L, et al. Arti ﬁcial neural\\nnetwork approach to predict surgical site infection after free- ﬂap recon-\\nstruction in patients receiving surgery for head and neck cancer. Oncotarget\\n2018;9:13768 e82.https://doi.org/10.18632/oncotarget.24468 .\\n[57] Ke C, Jin Y, Evans H, Lober B, Qian X, Liu J, et al. Prognostics of surgical site\\ninfections using dynamic health data. J Biomed Inform 2017;65:22 e33.\\nhttps://doi.org/10.1016/j.jbi.2016.10.021 .\\n[58] Rawson TM, Hernandez B, Moore LSP, Blandy O, Herrero P, Gilchrist M, et al.\\nSupervised machine learning for the prediction of infection on admission to\\nhospital: a prospective observational cohort study. J Antimicrob Chemother\\n2019;74:1108 e15.https://doi.org/10.1093/jac/dky514 .\\n[59] Horng S, Sontag DA, Halpern Y, Jernite Y, Shapiro NI, Nathanson LA. Creating\\nan automated trigger for sepsis clinical decision support at emergency\\ndepartment triage using machine learning. Groza T, ed. PLoS One 2017;12:\\ne0174708. https://doi.org/10.1371/journal.pone.0174708 .\\n[60] Taylor RA, Moore CL, Cheung K-H, Brandt C. Predicting urinary tract in-\\nfections in the emergency department with machine learning. PLoS One\\n2018;13:e0194085. https://doi.org/10.1371/journal.pone.0194085 .\\n[61] Revett K, Gorunescu F, Ene M. A machine learning approach to differenti-\\nating bacterial from viral meningitis. IEEE; 2006. p. 155 e62.https://doi.org/\\n10.1109/JVA.2006.2 .\\n[62] D ’Angelo G, Pilla R, Tascini C, Rampone S. A proposal for distinguishing be-\\ntween bacterial and viral meningitis using genetic programming and deci-sion trees. Soft Comput 2019. https://doi.org/10.1007/s00500-018-03729-y .\\n[63] Shamshirband S, Hessam S, Javidnia H, Amiribesheli M, Vahdat S, Petkovi /C19cD ,\\net al. Tuberculosis disease diagnosis using arti ﬁcial immune recognition\\nsystem. Int J Med Sci 2014;11:508 e14.https://doi.org/10.7150/ijms.8249 .\\n[64] Hatzakis GE, Mathur M, Gilbert L, Panos G, Wanchu A, Patel AK, et al. Neural\\nnetwork-longitudinal assessment of the Electronic Anti-Retroviral THerapy\\n(EARTH) cohort to follow response to HIV-treatment. AMIA Annu Symp ProcAMIA Symp 2005:301 e5\\n.\\n[65] Beerenwinkel N, Lengauer T, D €aumer M, Kaiser R, Walter H, Korn K, et al.\\nMethods for optimizing antiviral combination therapies. Bioinforma Oxf Engl\\n2003;19:i16 e25.\\n[66] Zazzi M, Incardona F, Rosen-Zvi M, Prosperi M, Lengauer T, Altmann A, et al.\\nPredicting response to antiretroviral treatment by machine learning: the\\nEuResist project. Intervirology 2012;55:123 e7.https://doi.org/10.1159/\\n000332008 .\\n[67] Revell AD, Wang D, Perez-Elias M-J, Wood R, Cogill D, Tempelman H, et al.\\n2018 update to the HIV-TRePS system: the development of new computa-tional models to predict HIV treatment outcomes, with or without a geno-\\ntype, with enhanced usability for low-income settings. J Antimicrob\\nChemother 2018;73:2186 e96.https://doi.org/10.1093/jac/dky179 .\\n[68] Petersen ML, LeDell E, Schwab J, Sarovar V, Gross R, Reynolds N, et al. Super\\nlearner analysis of electronic adherence data improves viral prediction and\\nmay provide strategies for selective HIV RNA monitoring. J Acquir Immune\\nDeﬁc Syndr 1999 2015;69:109 e18. https://doi.org/10.1097/QAI.0000000\\n000000548 .N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 594[69] Maiellaro PA, Cozzolongo R, Marino P. Arti ﬁcial neural networks for the\\nprediction of response to interferon plus ribavirin treatment in patients with\\nchronic hepatitis C. Curr Pharm Des 2004;10:2101 e9.\\n[70] Takayama T, Ebinuma H, Tada S, Yamagishi Y, Wakabayashi K, Ojiro K, et al.\\nPrediction of effect of pegylated interferon alpha-2b plus ribavirin combi-\\nnation therapy in patients with chronic hepatitis C infection. PLoS One2011;6:e27223. https://doi.org/10.1371/journal.pone.0027223 .\\n[71] Sargolzaee Aval F, Behnaz N, Raoufy MR, Alavian SM. Predicting the out-\\ncomes of combination therapy in patients with chronic hepatitis C using\\nartiﬁcial neural network. Hepat Mon 2014;14:e17028. https://doi.org/\\n10.5812/hepatmon.17028 .\\n[72] Hussain OA, Junejo KN. Predicting treatment outcome of drug-susceptible\\ntuberculosis patients using machine-learning models. Inform Health SocCare 2018:1 e17.https://doi.org/10.1080/17538157.2018.1433676 .\\n[73] Sauer CM, Sasson D, Paik KE, McCague N, Celi LA, S /C19anchez Fern /C19andez I, et al.\\nFeature selection and prediction of treatment failure in tuberculosis. Huang\\nZ, ed. PLoS One 2018;13:e0207491. https://doi.org/10.1371/journal.pone.\\n0207491 .\\n[74] Li BY, Oh J, Young VB, Rao K, Wiens J. Using machine learning and the\\nelectronic health record to predict complicated Clostridium dif ﬁcile infec-\\ntion. Open Forum Infect Dis 2019;6:ofz186. https://doi.org/10.1093/o ﬁd/\\nofz186 .\\n[75] LaBarbera FD, Nikiforov I, Parvathenani A, Pramil V, Gorrepati S. A prediction\\nmodel for Clostridium dif ﬁcile recurrence. J Community Hosp Intern Med\\nPerspect 2015;5:26033. https://doi.org/10.3402/jchimp.v5.26033 .\\n[76] Escobar GJ, Baker JM, Kipnis P, Greene JD, Mast TC, Gupta SB, et al. Prediction\\nof recurrent Clostridium dif ﬁcile infection using comprehensive electronic\\nmedical records in an integrated healthcare delivery system. Infect Control\\nHosp Epidemiol 2017;38:1196 e203. https://doi.org/10.1017/ice.2017.176 .\\n[77] Li S, Tang B, He H. An imbalanced learning based MDR-TB early warning\\nsystem. J Med Syst 2016;40. https://doi.org/10.1007/s10916-016-0517-2 .\\n[78] Yelin I, Snitser O, Novich G, Katz R, Tal O, Parizade M, et al. Personal clinical\\nhistory predicts antibiotic resistance of urinary tract infections. Nat Med\\n2019;25:1143 e52.https://doi.org/10.1038/s41591-019-0503-6 .\\n[79] Tacconelli E, G /C19orska A, De Angelis G, Lammens C, Restuccia G, Schrenzel J,\\net al. Estimating the association between antibiotic exposure and coloniza-\\ntion with extended-spectrum\\nb-lactamase-producing Gram-negative bacte-\\nria using machine learning methods: a multicentre, prospective cohort\\nstudy. Clin Microbiol Infect 2020;26:87 e94.\\n[80] Beaudoin M, Kabanza F, Nault V, Valiquette L. Evaluation of a machine\\nlearning capability for a clinical decision support system to enhance anti-\\nmicrobial stewardship programs. Artif Intell Med 2016;68:29 e36.https://\\ndoi.org/10.1016/j.artmed.2016.02.001 .\\n[81] Shen Y, Yuan K, Chen D, Colloc J, Yang M, Li Y, et al. An ontology-driven clinical\\ndecision support system (IDDAP) for infectious disease diagnosis and antibiotic\\nprescription. Artif Intell Med 2018;86:20 e32. https://doi.org/10.1016/\\nj.artmed.2018.01.003 .\\n[82] Ying Hao, Lin Feng, MacArthur Rodger D, Cohn Jonathan A, Barth-\\nJones Daniel C, Ye Hong, et al. A self-learning fuzzy discrete event system for\\nHIV/AIDS treatment regimen selection. IEEE Trans Syst MAN Cybern B\\nCybern 2007;37. https://doi.org/10.1109/TSMCB.2007.895360 .\\n[83] Gulshan V, Peng L, Coram M, Stumpe MC, Wu D, Narayanaswamy A, et al.\\nDevelopment and validation of a deep learning algorithm for detection of\\ndiabetic retinopathy in retinal fundus photographs. JAMA 2016;316:2402.https://doi.org/10.1001/jama.2016.17216 .\\n[84] Rajkomar A, Hardt M, Howell MD, Corrado G, Chin MH. Ensuring fairness in\\nmachine learning to advance health equity. Ann Intern Med 2018;169:866.\\nhttps://doi.org/10.7326/M18-1990 .\\n[85] Gianfrancesco MA, Tamang S, Yazdany J, Schmajuk G. Potential biases in ma-\\nchine learning algorithms using electronic health record data. JAMA Intern\\nMed 2018;178:1544 e7.https://doi.org/10.1001/jamainternmed.2018.3763 .\\n[86] Osakunor DNM, Sengeh DM, Mutapi F. Coinfections and comorbidities in\\nAfrican health systems: at the interface of infectious and noninfectious dis-\\neases. PLoS Negl Trop Dis 2018;12. https://doi.org/10.1371/\\njournal.pntd.0006711 .\\n[87] Taneja I, Reddy B, Damhorst G, Dave Zhao S, Hassan U, Price Z, et al. Combining\\nbiomarkers with EMR data to identify patients in different phases of sepsis. Sci\\nRep 2017;7:10800. https://doi.org/10.1038/s41598-017-09766-1 .[88] Ford E, Carroll JA, Smith HE, Scott D, Cassell JA. Extracting information from\\nthe text of electronic medical records to improve case detection: a system-\\natic review. J Am Med Inform Assoc 2016;23:1007 e15. https://doi.org/\\n10.1093/jamia/ocv180 .\\n[89] Greenhalgh T, Wherton J, Papoutsi C, Lynch J, Hughes G, A ’Court C, et al.\\nBeyond adoption: a new framework for theorizing and evaluating non-adoption, abandonment, and challenges to the scale-up, spread, and sus-\\ntainability of health and care technologies. J Med Internet Res 2017;19:e367.\\nhttps://doi.org/10.2196/jmir.8775 .\\n[90] Stead WW. Clinical implications and challenges of arti ﬁcial intelligence and\\ndeep learning. JAMA 2018;320:1107 e8. https://doi.org/10.1001/\\njama.2018.11029 .\\n[91] Rawson TM, Charani E, Moore LSP, Hernandez B, Castro-S /C19anchez E, Herrero P,\\net al. Mapping the decision pathways of acute infection management in sec-\\nondary care among UK medical physicians: a qualitative study. BMC Med\\n2016;14. https://doi.org/10.1186/s12916-016-0751-y0751-y .\\n[92] Maddox TM, Rumsfeld JS, Payne PRO. Questions for arti ﬁcial intelligence in\\nhealth care. JAMA 2018. https://doi.org/10.1001/jama.2018.18932 .\\n[93] He J, Baxter SL, Xu J, Xu J, Zhou X, Zhang K. The practical implementation of\\nartiﬁcial intelligence technologies in medicine. Nat Med 2019;25:30 e6.\\nhttps://doi.org/10.1038/s41591-018-0307-0 .\\n[94] Asan O, Smith P, Montague E. More screen time, less face time\\neimplications\\nfor EHR design. J Eval Clin Pract 2014;20:896 e901. https://doi.org/10.1111/\\njep.12182 .\\n[95] Margalit RS, Roter D, Dunevant MA, Larson S, Reis S. Electronic medical re-\\ncord use and physician epatient communication: an observational study of\\nIsraeli primary care encounters. Patient Educ Couns 2006;61:134 e41.\\nhttps://doi.org/10.1016/j.pec.2005.03.004 .\\n[96] Nemati S, Holder A, Razmi F, Stanley MD, Clifford GD, Buchman TG. An\\ninterpretable machine learning model for accurate prediction of sepsis in theICU. Crit Care Med 2018;46:547 e53. https://doi.org/10.1097/\\nCCM.0000000000002936 .\\n[97] Price WN, Cohen IG. Privacy in the age of medical big data. Nat Med 2019;25:\\n37e43.https://doi.org/10.1038/s41591-018-0272-7 .\\n[98] Calvert J, Hoffman J, Barton C, Shimabukuro D, Ries M, Chettipally U, et al.\\nCost and mortality impact of an algorithm-driven sepsis prediction system.\\nJ Med Econ 2017;20:646 e51. https://doi.org/10.1080/13696998.2017.\\n1307203 .\\n[99] Sinha M, Jupe J, Mack H, Coleman TP, Lawrence SM, Fraley SI. Emerging\\ntechnologies for molecular diagnosis of sepsis. Clin Microbiol Rev 2018;31.\\nhttps://doi.org/10.1128/CMR.00089-17 . e00089-17.\\n[100] Bisaso KR, Anguzu GT, Karungi SA, Kiragga A, Castelnuovo B. A survey of\\nmachine learning applications in HIV clinical research and care. Comput Biol\\nMed 2017;91:366 e71.https://doi.org/10.1016/j.compbiomed.2017.11.001 .\\n[101] Zazzi M, Cozzi-Lepri A, Prosperi MCF. Computer-aided optimization of\\ncombined anti-retroviral therapy for HIV: new drugs, new drug targets and\\ndrug resistance. Curr HIV Res 2016;14:101 e9.\\n[102] Al-Garadi MA, Khan MS, Varathan KD, Mujtaba G, Al-Kabsi AM. Using online\\nsocial networks to track a pandemic: a systematic review. J Biomed Inform\\n2016;62:1 e11.https://doi.org/10.1016/j.jbi.2016.05.005 .\\n[103] Nourani E, Khunjush F, Durmus ¸ S. Computational approaches for prediction\\nof pathogen-host protein-protein interactions. Front Microbiol 2015;6:94.\\nhttps://doi.org/10.3389/fmicb.2015.00094 .\\n[104] Zhou H, Jin J, Wong L. Progress in computational studies of host-pathogen\\ninteractions. J Bioinform Comput Biol 2013;11:1230001. https://doi.org/\\n10.1142/S0219720012300018 .\\n[105] Segal G. Identi ﬁcation of legionella effectors using bioinformatic approaches.\\nMethods Mol Biol Clifton NJ 2013;954:595 e602. https://doi.org/10.1007/\\n978-1-62703-161-5_37 .\\n[106] Ackerman ME, Barouch DH, Alter G. Systems serology for evaluation of HIV\\nvaccine trials. Immunol Rev 2017;275:262 e70. https://doi.org/10.1111/\\nimr.12503 .\\n[107] Ba /C21gcıU, Bray M, Caban J, Yao J, Mollura DJ. Computer-assisted detection of\\ninfectious lung diseases: a review. Comput Med Imaging Graph Off J Comput\\nMed Imaging Soc 2012;36:72 e84.https://doi.org/10.1016/j.compmedimag.\\n2011.06.002 .\\n[108] Laketa V. Microscopy in infectious disease research - imaging across scales.\\nJ Mol Biol 2018;430:2612 e25.https://doi.org/10.1016/j.jmb.2018.06.018 .N. Peiffer-Smadja et al. / Clinical Microbiology and Infection 26 (2020) 584 e595 595\""
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.9"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "b1b0c36e95623c0bd250d9361f1571d610d1bef21cde4345c242d5654c64f694"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
